---
title: "**The status of the microbial census: an update**"
bibliography: references.bib
output:
  pdf_document:
    includes:
      in_header: header.tex
csl: mbio.csl
fontsize: 11pt
geometry: margin=1.0in
---


```{r knitr_settings, eval=TRUE, echo=FALSE, cache=FALSE}
opts_chunk$set("tidy" = TRUE)
opts_chunk$set("echo" = FALSE)
opts_chunk$set("eval" = TRUE)
opts_chunk$set("warning" = FALSE)
opts_chunk$set("cache" = FALSE)

inline_hook <- function(x){
	print(x)
	if(is.numeric(x)){
		if(abs(x - round(x)) < .Machine$double.eps^0.5){
			paste(format(x,big.mark=',', digits=0, scientific=FALSE))
		} else {
			paste(format(x,big.mark=',', digits=2, nsmall=2, scientific=FALSE))
		}

	} else {
    	paste(x)      
	}
}
knitr::knit_hooks$set(inline=inline_hook)


year_of_analysis <- "2015"

bact <- read.table(file='data/process/bacteria.v123.metadata', header=T, row.names=1, stringsAsFactors=FALSE)
bact_year_deposited <- gsub("^(\\d\\d\\d\\d)-.*", "\\1", bact$date)

arch <- read.table(file='data/process/archaea.v123.metadata', header=T, row.names=1, stringsAsFactors=FALSE)
arch_year_deposited <- gsub("^(\\d\\d\\d\\d)-.*", "\\1", arch$date)
```
\vspace{35mm}


Patrick D. Schloss^1$\dagger$^, Rene Girard^2^, Thomas Martin^2^, Joshua Edwards^2^, and J. Cameron Thrash^2$\dagger$^

\vspace{40mm}

$\dagger$ To whom correspondence should be addressed: pschloss@umich.edu and thrashc@lsu.edu

1\. Department of Microbiology and Immunology, University of Michigan, Ann Arbor, MI 48109

2\. Department of Biological Sciences, Louisiana State University, Baton Rouge, LA 70803


\newpage
\linenumbers


## Abstract

Abstract goes here.


\newpage

## Importance

Importance goes here.



\newpage

## Introduction
In 1983, the full-length 16S rRNA gene sequence of *Escherichia coli* (accession J01695) was deposited into NCBI's GenBank making it the first of more than 10 million 16S rRNA gene sequences to be deposited into the database [@Brosius1978]. GenBank accessions represent nearly one-third of all sequences deposited in the database making it the best-represented gene. As Sanger sequencing has given way to to-called "next generation sequencing" technologies, hundreds of millions of 16S rRNA gene sequences have been deposited into the NCBI's Sequence Read Archive. The expansion in sequencing throughput and increased access to sequencing technology has allowed for more environments to be sequenced at a deeper coverage resulting in the identification of novel taxa. The ability to obtain sequence data from microorganisms without cultivation has radically altered our perspective of their role in nearly every environment from deep ocean sediment cores (e.g. accession AY436526) to the International Space Station (e.g. accession DQ497748). The effort to quantify the number of different organisms in a system remains fundamental to understanding ecology [@reference]. At the scale of microorganisms, small physical sizes, morphological ambiguity, and highly variable population sizes complicate this process. Furthermore, creating standards for delimiting what makes one microbe "different” from another has been contentious [@reference]. In spite of these challenges, we continue to peel back the curtain on the microbial world with the aid of more and more informative, if still limited, technologies like cultivation, 16S rRNA gene surveys, and metagenomics.

The increase in sequencing volume has come at the cost of sequence length. The commonly used MiSeq-based sequencing platform from Illumina is widely used to sequence the approximately 250 bp V4 hypervariable region of the 16S rRNA gene; other schemes have used different parts of the gene that are generally shorter than 500 bp. Perhaps most disconcerting about this development is the sense that the increased read depth is being gained using short read platforms rather than the full-length sequences. Because these short reads are used for classification to existing taxa, we are missing the opportunity to propose novel candidate taxa and vastly underappreciating the biodiversity of microbial life. We likely lack the references necessary to adequately classify the novel biodiversity we are sampling when we generate 100-times the sequence data from a community than we did using full-length sequencing.

Previously, Schloss and Handelsman [-@Schloss2004] assigned the then available 56,215 partial rRNA gene sequences to operational taxonomic units (OTUs) that were available in the Ribosomal Database Project and concluded that the sampling methods of the time were insufficient to identify the previously estimated 10^7^ to 10^9^ different species [@Dykhuizen1998, @Curtis2002]. That census called for a broader and deeper characterization of all environments. Refreshingly, this challenge was largely met. There have been major investments in studying the Earth's microbiome using 16S rRNA gene sequencing through initiatives such as the Human Microbiome Project [@reference], the Earth Microbiome Project [@reference], and the International Census of Marine Microorganisms. But most important, the original census was performed on the cusp of radical developments in sequencing technologies. That advancement has largely moved the bulk of sequencing throughput from large sequencing centers to individual investigators and leveraged their diversity to expand the representation of organisms and environments represented in public databases.

Here we update the status of the microbial census with nearly or completely full-length 16S rRNA gene sequences. In the 13 years since the collection of data for Schloss and Handelsman’s analysis, the number of full-length sequences has grown exponentially, despite the overwhelming contemporary focus by most researchers on short-read technologies. This update to the census allows us to evaluate the relative sampling thoroughness for different environments and clades, and make an argument for the continued need to collection full-length sequence data from many systems that have a long history of study. Although there has been a robust growth in the number of full-length sequences deposited to GenBank since its creation in 1983, the rate of growth has stalled over the past 5 years and the deposits have been dominated by a handful of research groups studying a limited number of environments. As researchers consider coalescing into a Unified Microbiome Initiative [@Alivisatos2015], it will be important to balance the need for mechanism-based studies with the need to generate full-length reference sequences from a diversity of environments. In this, continued technological advances, such as the reconstruction of nearly full-length sequences from metagenomics with EMIRGE (Miller 2011) and inovations in culturing (references needed: iChip, Epstein dormancy, Thrash 2015; Gifford 2014), will be vital.


## Materials and Methods

***Sequence data curation.***
The July 19, 2015 release of the ARB-formatted SILVA small subunit (SSU) reference database (SSU Ref v.123) was downloaded from http://www.arb-silva.de/fileadmin/silva_databases/release_123/ARB_files/SSURef_123_SILVA_19_07_15_opt.arb.tgz [@REF]. This release is based on the EMBL-EBI/ENA Release 123, which was released in March 2015. The SILVA curators identify potential SSU sequences using keyword searches and sequence-based search using RNAmmer (http://www.arb-silva.de/documentation/release-123/). The SILVA curators then screened the 7,168,241 resulting sequences based on a minimum length criteria (<300 nt), number of ambiguous base calls (>2%), length of sequence homopolymers (>2%), presence of vector contamination (>2%), low alignment quality value (<75), and likelihood of being chimeric (Pintail value < 50). Of the remaining sequences, the bacterial reference set retained those bacterial sequences longer than 1,200 nt and the archaeal reference set retained those archaeal sequences longer than 900 nt. The aligned 1,515,024 bacterial and 59,240 archaeal sequences were exported from the database using ARB along with the complete set of metadata. Additional sequence data was included from single-cell genomes available on the Integrated Microbial Genomes (IMG) system [@REF], many of which were recently obtained via the GEBA-MDM effort in Rinke et al. [@REF]. "SCGC” was searched on the IMG database March 12, 2015 to download the bacterial (N=249) and archaeal (N=46) 16S rRNA gene sequences and their associated metadata. The IMG sequences were aligned against the respective SILVA-based reference using mothur [@REF]. The aligned bacterial and archaeal sequence sets from SILVA and IMG were pooled and processed in parallel in mothur. Using mothur, sequences were further screened to remove any sequence with more than 2 ambiguous base calls and trimmed to overlap the same alignment coordinates while maintaining a minimum sequence length of 1,200 and 900 nt for the bacterial and archeael datasets, respectively. The final datasets contained `r nrow(bact)` bacterial and `r nrow(arch)` archaeal 16S rRNA gene sequences. Sequences were assigned to operational taxonomic units (OTUs) with a 3% distance threshold using the average neighbor clustering algorithm.


***Metadata curation.***
The environmental origins of the 16S rRNA gene sequences were manually classified using seven broad "coarse” categories, and further refined to facilitate additional analyses with twenty-six more specific "fine” categories (Table 1). These were assigned based on manual curation of the `isolation_source` category within the ARB database associated with each of the sequences. There were `r length(unique(bact$isolation_source))-1` bacterial and `r length(unique(arch$isolation_source))-1` archaeal isolation_source terms that were manually assigned to the coarse and fine categories in Table 1. For source definitions that were identifiable by online searches, educated guesses were made or they were placed into the coarse "Other” category. There were `r sum(is.na(bact$isolation_source))` bacterial and `r sum(is.na(arch$isolation_source))` archaeal sequences where an `isolation_source` term was not collected. Complete tables containing the ARB-provided metadata, taxonomic information, OTU assignment, and our categorization are available at [@figshare].


***Data analysis.***
Our analysis made use of ARB (OS X v.6.0) [@Pruesse2007], mothur (v.1.37.0) [@Schloss2009], and R (v.`r paste(version$major, version$minor, sep='.')`) [@language2015]. Within R we utilized knitr (v.`r packageDescription("knitr")$Version`) [@xie2013dynamic] and openxlsx (v. `r packageDescription("openxlsx")$Version`) [@Walker2015] packages. A reproducible version of this manuscript including data extraction and processing is available at https://www.github.com/SchlossLab/Schloss_Census2_mBio_2015.


## Results and Discussion

* What does the ongonig census look like?
  - flat
* Why might it be flat?
  - Good's coverage
  - Schloss's coverage
  - % of NGS reads that map to database
* Why aren't we doig a great job?
  - Rate of change in full-length sequences has plateaued
  - New sequences are submitted by a small number of investigators for a limited number of environments
* What can we do to fix this?
  - Characterize coverage by environments
  - Cultured vs. uncultured
  - Single cell genomics

Issues we sought to solve...

* Figure 1A. How have the number of full-length sequences changed over the past N years?
* Figure 1B. How does this vary by environment?

* Figure 2. How has the number of sequences per study changed over the past N years?

* Figure 3. How has the coverage of the OTUs represented by these sequences changed over the last N years?

* How has Figure 1 changed over the past N years?

* In what environments do we have the best handle on extant diversity?
  Figure 4A,B. What are the differences in sampling effort/sequence coverage at the 97% level for the different coarse habitats, and how is this different between Bacteria and Archaea?
  Figure 5A,B. For the most sampled coarse habitats (Zoonotic for Bacteria, Aquatic for Archaea), what are the fine-scale habitats where they have been sampled most heavily?

* In what habitats do we have the poorest understanding of diversity (and thus expect to yield considerable new knowledge over the years)?
* How is this different for Bacteria vs. Archaea?
* What are the most significant improvements since 2004, and where are we still lacking the most data?
* For which phyla has there been substantial increases in sampled diversity since 2004?
* Can we make any recommendations for habitats where biodiversity is under-sampled, perhaps calls for conservation in certain environments based on how poorly we understand the microbial diversity?


***Caveats.***
Recent data suggests that a considerable diversity of microorganisms may be missing based on biases in existing 16S rRNA gene primers. This dataset does not include sequences from metagenomic assemblies but Brown et al. (2015) have used such assemblies to show evidence for introns in the 16S rRNA genes of organisms within the so-called "Candidate Phyla Radiation” (CPR- Saccharibacteria (TM7), Peregrinibacteria, Berkelbacteria (ACD58), WWE3 Microgenomates (OP11), Parcubacteria (OD1), et al.), that would preclude detection with standard cultivation-independent microbial surveys. Furthermore, many of these CPR organisms are very small and frequently pass through 0.2 µm filters (Luef, 2015). Thus, for many environments, the estimates within must be considered as lower bounds.

Generating a comprehensive understanding of any system with a single gene may seem a fool’s errand, yet we have learned a considerable amount regarding the diversity, dynamics, and natural history of microorganisms using the venerable 16S rRNA gene. Indeed, continual community efforts to obtain 16S rRNA gene assessments of every environment possible have presented us with an ever-increasing estimate of total microbial diversity and the concomitant excitement of frontier science. While reliance on this gene subjects us to biases created by primer selection [REFS], differences in amplification strength [REFS] and fidelity [REFS], internal features which may disrupt traditional measurements [REFS], and potentially misleading classification due to infrequent horizontal gene transfer [REFS], the total data available from persistent collection of 16S rRNA gene sequences nevertheless dwarfs that of any other genetic marker. Thus, an attempt to quantify how much of the microbial world has been revealed inevitably starts there.



## Conclusions

Future for PacBio in generating full-length sequences

The first 16S sequence was published in 1978, not deposited until 1983. A bit of an allegory for our time.

Impact of EMIRGE

Renewed call for cultivation

\newpage
\singlespacing



**Figure 1. Sequences deposited by year**

Pat, this is a placeholder for you

**Figure 2. Sequence deposition by study**

Pat, ibid.

**Figure 3. Sampling by OTU cutoff**

I only have an old code from Rene for this, and I don't think it reports all the sequences


**Figure 4A. Bacterial sampling by environment**

```{r figure4a}
data <- read.table(file="data/mothur/all_bacteria.env_category.groups.rarefaction", header=T)
category <- "\\.plant$" #change the SO to whatever category you want (leave the \\. and $)
plant_columns <- grep(category, colnames(data))
category <- "\\.aerosol$" #change the SO to whatever category you want (leave the \\. and $)
aerosol_columns <- grep(category, colnames(data))
category <- "\\.aquatic$" #change the SO to whatever category you want (leave the \\. and $)
aquatic_columns <- grep(category, colnames(data))
category <- "\\.built$" #change the SO to whatever category you want (leave the \\. and $)
built_columns <- grep(category, colnames(data))
category <- "\\.other$" #change the SO to whatever category you want (leave the \\. and $)
other_columns <- grep(category, colnames(data))
category <- "\\.soil$" #change the SO to whatever category you want (leave the \\. and $)
soil_columns <- grep(category, colnames(data))
category <- "\\.zoonotic$" #change the SO to whatever category you want (leave the \\. and $)
zoonotic_columns <- grep(category, colnames(data))
plant_aerosol_aquatic_built_other_soil_zoonotic_data <- data[,c(1, plant_columns, aerosol_columns, aquatic_columns, built_columns, other_columns, soil_columns, zoonotic_columns)]  #the 1 gets the first column which is numsampled
nrows_to_keep <- which.max(plant_aerosol_aquatic_built_other_soil_zoonotic_data$numsampled[!is.na(plant_aerosol_aquatic_built_other_soil_zoonotic_data[,2])])
plant_aerosol_aquatic_built_other_soil_zoonotic_data <- plant_aerosol_aquatic_built_other_soil_zoonotic_data[1:nrows_to_keep,]
pdf(file="/Users/renegirard7gfy/Desktop/Bac_Uniq_3per.pdf")

par(mar=c(5,5,0.5,0.5))
plot(NA, type="n", ylim=c(0,max(40000)),
     xlim=c(0,max(data$numsampled)),
     xlab="Number of sequences sampled",
     ylab="",
     axes=FALSE)

points(data$X0.03.plant~data$numsampled, type="l", col="red", lwd=2)
points(data$X0.03.aerosol~data$numsampled, type="l", col="green", lwd=2)
points(data$X0.03.aquatic~data$numsampled, type="l", col="blue", lwd=2)
points(data$X0.03.built~data$numsampled, type="l", col="orange", lwd=2)
points(data$X0.03.other~data$numsampled, type="l", col="black", lwd=2)
points(data$X0.03.soil~data$numsampled, type="l", col="purple", lwd=2)
points(data$X0.03.zoonotic~data$numsampled, type="l", col="deeppink", lwd=2)

legend(x=5e5, y=20000, lwd=2, lty=1,
       legend=c("Other", "Plant", "Aerosol", "Aquatic", "Built", "Soil", "Zoonotic"),
       col=c("black", "red", "green", "blue", "orange", "purple", "deeppink"))

axis(1, at=seq(0,700000,1e5), label=c("0", "100,000", "200,000", "300,000", "400,000", "500,000", "600,000", "700,000"), cex.axis=0.8)
axis(2, at=seq(0,40000,5000), label=c("0", "5,000", "10,000", "15,000", "20,000", "25,000", "30,000", "35,000", "40,000"), las=2, cex.axis=0.8)
box()
mtext(side=2, "Number of OTUs", line=4)

dev.off()

```

**Figure 4B. Archaeal sampling by environment**

```{r figure4b}

data <- read.table(file="data/mothur/all_archaea.env_category.groups.rarefaction", header=T)
category <- "\\.plant$" #change the SO to whatever category you want (leave the \\. and $)
plant_columns <- grep(category, colnames(data))
category <- "\\.aerosol$" #change the SO to whatever category you want (leave the \\. and $)
aerosol_columns <- grep(category, colnames(data))
category <- "\\.aquatic$" #change the SO to whatever category you want (leave the \\. and $)
aquatic_columns <- grep(category, colnames(data))
category <- "\\.built$" #change the SO to whatever category you want (leave the \\. and $)
built_columns <- grep(category, colnames(data))
category <- "\\.other$" #change the SO to whatever category you want (leave the \\. and $)
other_columns <- grep(category, colnames(data))
category <- "\\.soil$" #change the SO to whatever category you want (leave the \\. and $)
soil_columns <- grep(category, colnames(data))
category <- "\\.zoonotic$" #change the SO to whatever category you want (leave the \\. and $)
zoonotic_columns <- grep(category, colnames(data))
plant_aerosol_aquatic_built_other_soil_zoonotic_data <- data[,c(1, plant_columns, aerosol_columns, aquatic_columns, built_columns, other_columns, soil_columns, zoonotic_columns)]  #the 1 gets the first column which is numsampled
nrows_to_keep <- which.max(plant_aerosol_aquatic_built_other_soil_zoonotic_data$numsampled[!is.na(plant_aerosol_aquatic_built_other_soil_zoonotic_data[,2])])
plant_aerosol_aquatic_built_other_soil_zoonotic_data <- plant_aerosol_aquatic_built_other_soil_zoonotic_data[1:nrows_to_keep,]
pdf(file="/Users/renegirard7gfy/Desktop/Arc_Uniq_3per.pdf")

par(mar=c(5,5,0.5,0.5))
plot(NA, type="n", ylim=c(0,max(data[,-1], na.rm=T)),
     xlim=c(0,max(data$numsampled)),
     xlab="Number of sequences sampled",
     ylab="",
     axes=FALSE)

points(data$X0.03.plant~data$numsampled, type="l", col="red", lwd=2)
points(data$X0.03.aerosol~data$numsampled, type="l", col="green", lwd=2)
points(data$X0.03.aquatic~data$numsampled, type="l", col="blue", lwd=2)
points(data$X0.03.built~data$numsampled, type="l", col="orange", lwd=2)
points(data$X0.03.other~data$numsampled, type="l", col="black", lwd=2)
points(data$X0.03.soil~data$numsampled, type="l", col="purple", lwd=2)
points(data$X0.03.zoonotic~data$numsampled, type="l", col="deeppink", lwd=2)

legend(x=15000, y=1550, lwd=2, lty=1,
       legend=c("Other", "Plant", "Aerosol", "Aquatic", "Built", "Soil", "Zoonotic"),
       col=c("black", "red", "green", "blue", "orange", "purple", "deeppink"))

axis(1, at=seq(0,35000,5000), label=c("0", "5,000", "10,000", "15,000", "20,000", "25,000", "30,000", "35,000"), cex.axis=0.8)
axis(2, at=seq(0,5250,750), label=c("0", "750", "1500", "2250", "3,000", "3750", "4500", "5250"), las=2, cex.axis=0.8)
box()
mtext(side=2, "Number of OTUs", line=4)

dev.off()
```

\newpage
\doublespacing


## References
