---
title: "**The status of the microbial census: an update**"
bibliography: references.bib
output:
  pdf_document:
    includes:
      in_header: header.tex
csl: mbio.csl
fontsize: 11pt
geometry: margin=1.0in
---


```{r knitr_settings, eval=TRUE, echo=FALSE, cache=FALSE}
opts_chunk$set("tidy" = TRUE)
opts_chunk$set("echo" = FALSE)
opts_chunk$set("eval" = TRUE)
opts_chunk$set("warning" = FALSE)
opts_chunk$set("cache" = FALSE)

inline_hook <- function(x){
	print(x)
	if(is.numeric(x)){
		if(abs(x - round(x)) < .Machine$double.eps^0.5){
			paste(format(x,big.mark=',', digits=0, scientific=FALSE))
		} else {
			paste(format(x,big.mark=',', digits=1, nsmall=1, scientific=FALSE))
		}

	} else {
    	paste(x)      
	}
}
knitr::knit_hooks$set(inline=inline_hook)

```
\vspace{35mm}

Running title: The microbial census

\vspace{35mm}


Patrick D. Schloss^1$\dagger$^, Rene Girard^2^, Thomas Martin^2^, Joshua Edwards^2^, and J. Cameron Thrash^2$\dagger$^

\vspace{40mm}

$\dagger$ To whom correspondence should be addressed: pschloss@umich.edu and thrashc@lsu.edu

1\. Department of Microbiology and Immunology, University of Michigan, Ann Arbor, MI 48109

2\. Department of Biological Sciences, Louisiana State University, Baton Rouge, LA 70803


\newpage
\linenumbers


## Abstract

A census is a useful tool for assessing the characteristics of an entity, determining the best distribution of resources across individuals, and highlighting the strengths and weaknesses of the entity. Although a census is typically carried out for people at a national level, microbial ecologists have carried out a molecular census of bacteria and archaea by sequencing their 16S rRNA genes. The goal of this study was to update an earlier effort from 2004 by assessing how well the microbial census of full-length 16S rRNA gene sequences is proceeding in the context of recent advances in high throughput sequencing technologies. We found that nearly all full-length sequences belong to OTUs that have been previously observed, but that most bacterial and archaeal operational taxonomic units (OTUs) have only been observed once. This suggests that there is still considerable microbial diversity to be explored. We discuss the role that traditional culture-dependent and -independent approaches have had in advancing the census as well as the possible role that new technologies such as single cell genomics and short read assembly might have in allowing researchers to improve our ability to sample rare OTUs. We also show that environments that are known to be rich in bacterial diversity (e.g. soil and aquatic environments) have been poorly sampled. The ability to contextualize the biodiversity that is sampled using short read sequencing technologies depends on the ability to obtain similar sequence depths of full-length sequences from diverse environments. Furthermore, these sequences and their environmental metadata must be made available in public databases.


### Importance

The biodiversity contained within the bacterial and archaeal domains dwarfs that of the eukaryotes and the services these organisms provide to the biosphere are critical. Surprisingly, we have done a relatively poor job of keeping track of the ongoing effort to characterize the biodiversity as represented in full-length 16S rRNA genes. By understanding how this census is proceeding, it is possible to suggest the best allocation of resources for advancing the census. We actually found that the ongoing effort has done an excellent job of sampling the most abundant organisms, but struggles to sample the more rare organisms. Through the use of new sequencing technologies we should be able to obtain full-length sequences from these rare organisms. Furthermore, we suggest that by allocating more resources to sampling environments known to have the greatest biodiversity we will be able to make significant advances in our characterization of microbial diversity.


\newpage

## Introduction
The effort to quantify the number of different organisms in a system remains fundamental to understanding ecology [@reference]. At the scale of microorganisms, small physical sizes, morphological ambiguity, and highly variable population sizes complicate this process. Furthermore, creating standards for delimiting what makes one microbe "different” from another has been contentious [@reference]. In spite of these challenges, we continue to peel back the curtain on the microbial world with the aid of more and more informative, if still limited, technologies like cultivation, 16S rRNA gene surveys, single cell technologies, and metagenomics.

Generating a comprehensive understanding of any system with a single gene may seem a fool’s errand, yet we have learned a considerable amount regarding the diversity, dynamics, and natural history of microorganisms using the venerable 16S rRNA gene. In 1983, the full-length 16S rRNA gene sequence of *Escherichia coli* (accession J01695) was deposited into NCBI's GenBank making it the first of what is now more than 10 million 16S rRNA gene sequences to be deposited into the database [@Brosius1978]. 16S rRNA gene accessions represent nearly one-third of all sequences deposited in GenBank, making it the best-represented gene. As Sanger sequencing has given way to so-called "next generation sequencing" technologies, hundreds of millions of 16S rRNA gene sequences have been deposited into the NCBI's Sequence Read Archive. The expansion in sequencing throughput and increased access to sequencing technology has allowed for more environments to be sequenced at a deeper coverage, resulting in the identification of novel taxa. The ability to obtain sequence data from microorganisms without cultivation has radically altered our perspective of their role in nearly every environment from deep ocean sediment cores (e.g. accession AY436526) to the International Space Station (e.g. accession DQ497748).

Previously, Schloss and Handelsman [-@Schloss2004] assigned the then available 56,215 partial 16S rRNA gene sequences to operational taxonomic units (OTUs) that were available in the Ribosomal Database Project and concluded that the sampling methods of the time were insufficient to identify the previously estimated 10^7^ to 10^9^ different species [@Dykhuizen1998, @Curtis2002]. That census called for a broader and deeper characterization of all environments. Refreshingly, this challenge was largely met. There have been major investments in studying the Earth's microbiome using 16S rRNA gene sequencing through initiatives such as the Human Microbiome Project [@reference], the Earth Microbiome Project [@reference], and the International Census of Marine Microorganisms. But most importantly, the original census was performed on the cusp of radical developments in sequencing technologies. That advancement has moved the generation of sequencing throughput from large sequencing centers to individual investigators and leveraged their diverse interests to expand the representation of organisms and environments represented in public databases.

Disconcertingly, the increase in sequencing volume has come at the cost of sequence length. The commonly used MiSeq-based sequencing platform from Illumina is extensively leveraged to sequence the approximately 250 bp V4 hypervariable region of the 16S rRNA gene; other schemes have used different parts of the gene that are generally shorter than 500 bp. Youssef et al. (2009) demonstrated high variability between the number of OTUs that different hypervariable sites estimated compared to full-length 16S rRNA gene sequences, and also that sample location influenced this behavior. Thus, it remains unclear to what degree richness estimates from short read technology over or underestimate the numbers from full-length sequences. We likely lack the references necessary to adequately classify the novel biodiversity we are sampling when we generate 100-times the sequence data from a community than we did using full-length sequencing.

Here we update the status of the microbial census with nearly or completely full-length 16S rRNA gene sequences. In the 13 years since the collection of data for Schloss and Handelsman’s analysis, the number of full-length sequences has grown exponentially, despite the overwhelming contemporary focus by most researchers on short-read technologies. This update to the census allows us to evaluate the relative sampling thoroughness for different environments and clades, and make an argument for the continued need to collection full-length sequence data from many systems that have a long history of study. Although there has been a robust growth in the number of full-length sequences deposited to GenBank since its creation in 1983, the rate of growth has stalled over the past 5 years and the deposits have been dominated by a handful of research groups studying a limited number of environments. As researchers consider coalescing into a Unified Microbiome Initiative [@Alivisatos2015], it will be important to balance the need for mechanism-based studies with the need to generate full-length reference sequences from a diversity of environments. In this, continued technological advances, such as the reconstruction of nearly full-length sequences from metagenomics with EMIRGE (Miller 2011) and innovations in culturing (references needed: iChip, Epstein dormancy, Thrash 2015; Gifford 2014), will be vital.


## Results and Discussion

```{r rarefaction}
# take these data out of the rarefaction files so that we can have the unique,
# 0.03, 0.05, 0.10, and 0.20 OTU definitions for both the bacterial and archaeal
# data

b_sobs <- read.table(file="data/process/bacteria.all_rarefaction.tsv", header=T)
b_sobs <- b_sobs[nrow(b_sobs),]
b_nseqs <- as.numeric(rownames(b_sobs))

a_sobs <- read.table(file="data/process/archaea.all_rarefaction.tsv", header=T)
a_sobs <- a_sobs[nrow(a_sobs),]
a_nseqs <- as.numeric(rownames(a_sobs))


coverage_by_category_and_time <- read.table(file="data/process/coverage_by_category_and_time.tsv", header=T, stringsAsFactors=FALSE)

b_nseqs <- coverage_by_category_and_time["total", "bact_count"]
b_goods <- coverage_by_category_and_time["total", "bact_good_coverage"]
b_schloss <- coverage_by_category_and_time["total", "bact_schloss_coverage"]

a_nseqs <- coverage_by_category_and_time["total", "arch_count"]
a_goods <- coverage_by_category_and_time["total", "arch_good_coverage"]
a_schloss <- coverage_by_category_and_time["total", "arch_schloss_coverage"]
```

***The status of the bacterial and archaeal census.***
To assess the field's progress in characterizing the biodiversity of bacteria and archaea we assigned each 16S rRNA gene sequence to OTUs using distance threshold varying between 0 and 20%. Although it is not possible to link a specific taxonomic level (e.g. species, genus, family, etc.) to a specific distance threshold, we selected distances of 0, 3, 5, 10, and 20% because they are widely regarded as representing the range of genetic diversity of the 16S rRNA gene within each domain. By rarefaction, it was clear that the ongoing sampling efforts have started to saturate the number of current OTUs. After sampling `r b_nseqs` near full-length bacterial 16S rRNA gene sequences we have identified `r b_sobs['X0.00']`, `r b_sobs['X0.03']`, `r b_sobs['X0.05']`, `r b_sobs['X0.10']`, and `r b_sobs['X0.20']` OTUs at the respective thresholds (Figure 1A, Table 1). Using only the OTUs generated using a 3% threshold, we calculated a `r b_goods`% Good's coverage (percent of sequences belonging to OTUs that have been observed more than once), but only `r b_schloss`% OTU coverage (percent of the OTUs that have been observed more than once). Paralleling the bacterial results, after sampling `r a_nseqs` archaeal 16S rRNA gene sequences we have identified `r a_sobs['X0.00']`, `r a_sobs['X0.03']`, `r a_sobs['X0.05']`, `r a_sobs['X0.10']`, and `r a_sobs['X0.20']` OTUs (Figure 1B, Table 1). Using only the OTUs generated using a 3% threshold, we calculated a `r a_goods`% Good's coverage, but only `r a_schloss`% OTU coverage. These results indicate that regardless of the domain, continued sampling with the current strategies for generating full-length sequences will largely reveal OTUs that have already been observed, even though a large fraction of OTUs have only been sampled once. Furthermore, considering more than `r 100-min(b_schloss, a_schloss)`% of the OTUs have only been observed once, it is likely that an even larger number of OTUs have yet to be sampled for both domains.


```{r time_course}
time_data <- read.table(file="data/process/by_year_analysis.tsv", header=TRUE, stringsAsFactors=FALSE)

all_nseqs_2009_2012 <- sum(time_data[as.character(2009:2012), 'b_nseqs'])
dominant_nseqs_2009_2012 <- sum(time_data[as.character(2009:2012), 'b_n50_nseqs'])

# what were the papers that these sequences came from?
dominant_pmids_2009_2012 <- unique(unlist(strsplit(time_data[as.character(2009:2012), "b_n50_doi"], split=',')))

#require(knitcitations, quiet=TRUE)
#citep(c('10.1038/ismej.2012.79', '10.1371/journal.pone.0026284', '10.1101/gr.131029.111', '10.1073/pnas.1004204107', '10.1126/science.1171700'))

stopifnot(dominant_pmids_2009_2012 == c("19478181", "22310478", "20668241", "22719818", "22832344"))
#if this stops then will need to update stuff below
```

***The status of the bacterial census.***
One explanation for the large number of OTUs that have only been observed once is that with the the broad adoption of highly parallelized sequencing platforms that generate short sequence reads, the rate of full-length sequence generation has declined. In fact, since 2009 the number of new bacterial sequences generated has slowed two an average of `r floor(mean(time_data[as.character(2009:2014),'b_nseqs']))` sequences per year (Figure 2A). Although this is still an impressive number of sequences, since `r start<-2007;format(start, big.mark='')` the number of new bacterial OTUs has plateaued at an average of `r round(mean(time_data[as.character(start:2014),'b_sobs']),0)` new OTUs per year (Figure 2B). Given the expense of generating full-length sequences using the Sanger sequencing technology and the transition to other platforms at that time, we expected that the large number of sequences were being deposited by a handful of large projects. Indeed, when we counted the number of submissions responsible for depositing 50% of the sequences, we found that with the exception of 2006 and 2013, eight or fewer studies were responsible for depositing the majority of the full-length sequences each year since 2005 (Figure 2C). Between 2009 and 2012, `r all_nseqs_2009_2012` total sequences were submitted and `r sum(time_data[as.character(2009:2012), 'b_n50'])` submissions from `r length(dominant_pmids_2009_2012)` studies were responsible for depositing `r dominant_nseqs_2009_2012` (`r 100*dominant_nseqs_2009_2012/all_nseqs_2009_2012`% of all sequences). These studies generated sequences from the human gastrointestinal tract [@Li_2012], human skin [@Kong_2012; @Grice_2009], murine skin [@Grice_2010], and hypersaline microbial mats [@Kirk_Harris_2012]. The heavy zoonotic focus is reflected in the rarefaction curve for this category (Figure 1C). In contrast to recent years, between 1995 and 2006, an average of `r mean(time_data[as.character(1995:2006), 'b_n50'])` studies were responsible for submitting more than half of the sequences each year. Although these deep surveys represent significant contributions to our knowledge of bacterial biogeography, their small number and lack of environmental diversity is indicative of the broader problems in advancing the bacterial census.


***The status of the archaeal census.***
The depth of sequencing being done to advance the archaeal census has been `r floor(sum(time_data$b_nseqs)/sum(time_data$a_nseqs))`-times less than that of the bacterial census (Table 1). The annual number of sequences submitted has largely paralleled that of the bacterial census with a plateau starting in 2009 and an average of `r mean(time_data[as.character(2009:2014),'a_nseqs'])` sequences each year since then. The number of new archaeal OTUs represented by these sequences began to slow in 2005 with an average of `r mean(time_data[as.character(2005:2014),'a_sobs'])` new OTUs per year. With the exception of 2012 and 2014, the number of submissions responsible for more than 50% of the archaeal sequences submitted per year has varied between `r min(time_data[as.character(c(1996:2011,2013)),'a_n50'])` and `r max(time_data[as.character(c(1996:2011,2013)),'a_n50'])` submissions per year. The clear bias towards sequencing bacterial 16S rRNA genes has limited the ability to more fully characterize the biodiversity of the archaea, which is clearly reflected in the relatively meager sampling effort across habitats, compared to bacteria (Figure 1D),

```{r taxon_census}
oxford_comma <- function(ids){
	if(length(ids)==0){
		"ERROR"
	} else if(length(ids) == 1){
		ids
	} else if(length(ids) == 2){
		paste(ids, collapse=" and ")
	} else {
		paste(paste(ids[1:(length(ids)-1)], collapse=", "), ids[length(ids)], sep=", and ")
	}
}

b_phylum <- read.table(file="data/process/bacteria.phyla.counts.tsv", header=T)
rownames(b_phylum) <- gsub("Candidate_division_", "", rownames(b_phylum))
rownames(b_phylum) <- gsub("_", " ", rownames(b_phylum))

b_new_phyla <- rownames(b_phylum)[b_phylum$pre == 0]
b_new_phyla_names <- oxford_comma(b_new_phyla)


a_phylum <- read.table(file="data/process/archaea.phyla.counts.tsv", header=T)
a_new_phyla <- rownames(a_phylum)[a_phylum$pre == 0]

a_new_phyla_names <- oxford_comma(a_new_phyla)
a_new_phyla_names <- gsub("\\(\\S*\\)", "", a_new_phyla_names)
a_new_phyla_names <- gsub("_", " ", a_new_phyla_names)
```

***The ability to sample microbial life is taxonomically skewed.***
The Firmicutes, Proteobacteria, Actinobacteria, and Bacteroidetes represent `r 100*sum(b_phylum[c("Firmicutes", "Proteobacteria", "Actinobacteria", "Bacteroidetes"), "all"])/sum(b_phylum$all)`% of the bacterial sequences and the Euryarchaeota and Thaumarchaeota `r 100*sum(a_phylum[c("Euryarchaeota",  "Thaumarchaeota"), "all"])/sum(a_phylum$all)`% of the archaeal sequences. We sought to understand how the representation of individual phyla has changed relative to the state of the census in 2006. We used 2006 as a reference point for calibrating the dynamics of the bacterial and archaeal censuses since that was the year that the first highly parallelized 16S rRNA gene sequence dataset was published and ushered in a radical change in how microbial communities are studied [@Sogin2006]. In 2006 there were `r sum(b_phylum$pre!=0)` bacterial and `r sum(a_phylum$pre!=0)` phyla. Since then there have been `r length(b_new_phyla)` new  bacterial (`r b_new_phyla_names`) and `r length(a_new_phyla)` new archaeal candidate phyla (`r a_new_phyla_names`). Relative to the overall sequencing trends before and after 2006, several phyla stand out for being over and underrepresented in sequence submissions (Figure 3). Among the bacterial phyla with at least 1,000 sequences, `r oxford_comma(sort(rownames(b_phylum[b_phylum$ratio_of_ratios >= 4.0 & b_phylum$all >=1000,])))` were sequenced 4-fold more often while `r oxford_comma(sort(rownames(b_phylum[b_phylum$ratio_of_ratios < 0.5 & b_phylum$all >=1000,])))` were sequenced 2-fold less often than would have been expected since 2006. Among the archaeal phyla with at least 1,000 sequences, the Thaumarchaeota were sequenced `r a_phylum["Thaumarchaeota", "ratio_of_ratios"]`-fold more often and the Crenarchaeota were sequenced `r 1/a_phylum["Crenarchaeota", "ratio_of_ratios"]`-fold less often than expected. Together, these results demonstrate a change in the phylum-level lineages represented in the census from before and after 2006.



```{r environment}
strip <- function(x){	as.numeric(gsub(',', '', x))	}

time <- grepl('_pre', colnames(coverage_by_category_and_time)) | grepl('_post', colnames(coverage_by_category_and_time))
coverage_by_category <- coverage_by_category_and_time[,!time]

totals <- unlist(coverage_by_category[nrow(coverage_by_category),])
coverage_by_category <- coverage_by_category[-nrow(coverage_by_category),]

b_class_percent <- 100*sum(coverage_by_category[-nrow(coverage_by_category), "bact_count"])/sum(totals["bact_count"])

a_class_percent <- 100*sum(coverage_by_category[-nrow(coverage_by_category), "arch_count"])/sum(totals["arch_count"])


coarse_map <- c("A"=="aerosol", "AQ" = "aquatic", "B" = "built environment", "O" = "other", "P" = "plant", "S" = "soil", "Z" = "zoonotic")

coarse <- gsub("^(.)[A-Z]$", "\\1", rownames(coverage_by_category))
coarse <- gsub("^(..)[A-Z]+$", "\\1", coarse)

b_agg_count <- aggregate(coverage_by_category$bact_count, by=list(coarse_map[coarse]), sum)
b_text <- paste0(b_agg_count$Group.1, ' (N=', format(b_agg_count$x, big.mark=',', trim=TRUE), ')')[order(b_agg_count$x, decreasing=T)]

a_agg_count <- aggregate(coverage_by_category$arch_count, by=list(coarse_map[coarse]), sum)
a_text <- paste0(a_agg_count$Group.1, ' (N=', format(a_agg_count$x, big.mark=',', trim=TRUE), ')')[order(a_agg_count$x, decreasing=T)]


phylum_category <- read.table(file="data/process/phylum_category_counts.tsv")
bacteria <- phylum_category[phylum_category$domain == "bacteria", -1]

dominant_bacteria <- sort(table(apply(bacteria, 2, which.max)), decreasing=T)
names(dominant_bacteria) <- rownames(bacteria)[as.numeric(names(dominant_bacteria))]
dominant_bacteria_list <- oxford_comma(paste0(names(dominant_bacteria), ' (N=', dominant_bacteria, ')'))

archaea <- phylum_category[phylum_category$domain == "archaea", -1]
dominant_archaea <- sort(table(apply(archaea, 2, function(x){z<-which.max(x);if(length(z)==0){z<-NA};z})), decreasing=T)
names(dominant_archaea) <- rownames(archaea)[as.numeric(names(dominant_archaea))]
dominant_archaea_list <- oxford_comma(paste0(names(dominant_archaea), ' (N=', dominant_archaea, ')'))
```

***Focusing the census by environment.***
We were able to assign `r b_class_percent` and `r a_class_percent`% of the sequences to one of seven broad environmental categories based on the metadata that accompanied the SILVA database (Table S1). Across these broad categories there was wide variation in the number of sequences that have been sampled. Among bacterial sequences, the three best represented groups were from `r oxford_comma(b_text[1:3])` sources. Among the archaeal sequences the three best represented groups were the same, but ordered differently: `r oxford_comma(a_text[1:3])` (Figure 1C,D)). For both domains, soil samples were the fourth most represented category (bacteria: `r b_agg_count[b_agg_count$Group.1=="soil", "x"]`; archaea: `r a_agg_count[a_agg_count$Group.1=="soil", "x"]`). The orders of these categories was surprising considering soil and aquatic environments harbor the most microbial biomass and biodiversity [@Whitman1998]. In spite of wide variation in sequencing depth and coverage (Table 1), the interquartile range across the fine-level categories for the bacterial OTU coverage only varied between `r quantile(coverage_by_category$bact_schloss_coverage, prob=c(0.25))` to `r quantile(coverage_by_category$bact_schloss_coverage, prob=c(0.75))` (median coverage=`r median(coverage_by_category$bact_schloss_coverage, na.rm=T)`%). The interquartile range in the OTU coverage by environment for the archaeal data was `r quantile(coverage_by_category$arch_schloss_coverage, prob=c(0.25), na.rm=T)` to`r quantile(coverage_by_category$arch_schloss_coverage, prob=c(0.75), na.rm=T)` (median coverage=`r median(coverage_by_category$arch_schloss_coverage, na.rm=T)`%). The archaeal coverage was higher than that of the bacterial OTU coverage for all categories except the food-associated, plant surface, and other invertebrate categories. Across all categories, the bacterial and archaeal sequencing data represented a limited number of phyla (Figure 4). Among the bacterial data, the fine-scale categories were dominated by `r dominant_bacteria_list` and among the archaeal data, they were dominated by `r dominant_archaea_list`. Regardless, there were clear phylum-level signatures that differentiated the various categories. Within each of the bacterial and archaeal phyla, there was considerable variation in the relative abundance of each across the categories confirming that taxonomic signatures exist to differentiate different environments even at a broad taxonomic level.


```{r culture}
b_cult_time <- read.table(file='data/process/bacteria.cultured_by_time_counts.tsv', header=T)
a_cult_time <- read.table(file='data/process/archaea.cultured_by_time_counts.tsv', header=T)

b_percent_cult_nseqs_all <- 100*sum(b_cult_time[,"all_cult_nseqs"])/sum(b_cult_time[,"all_total_nseqs"])
a_percent_cult_nseqs_all <- 100*sum(a_cult_time[,"all_cult_nseqs"])/sum(a_cult_time[,"all_total_nseqs"])

b_shift_cult_nseqs_all <- 1/((sum(b_cult_time[,"post_cult_nseqs"])/sum(b_cult_time[,"pre_cult_nseqs"])) / (sum(b_cult_time[,"post_total_nseqs"])/sum(b_cult_time[,"pre_total_nseqs"])))

a_shift_cult_nseqs_all <- 1/((sum(a_cult_time[,"post_cult_nseqs"])/sum(a_cult_time[,"pre_cult_nseqs"])) / (sum(a_cult_time[,"post_total_nseqs"])/sum(a_cult_time[,"pre_total_nseqs"])))

b_rate_cult_nseqs_phyulm <- b_cult_time[, 'all_cult_nseqs'] / b_cult_time[, 'all_total_nseqs']
a_rate_cult_nseqs_phyulm <- a_cult_time[, 'all_cult_nseqs'] / a_cult_time[, 'all_total_nseqs']

b_percent_cult_otu_all <- 100*sum(b_cult_time[,'all_cult_sobs'])/sum(b_cult_time[,'all_total_sobs'])
a_percent_cult_otu_all <- 100*sum(a_cult_time[,'all_cult_sobs'])/sum(a_cult_time[,'all_total_sobs'])
```

***The cultured census.***
In the 2004 bacterial census there was great concern that although culture-independent methods were significantly enhancing our knowledge of microbial life, there were numerous bacterial phyla with no or only a few cultured representatives. To update this assessment, we identified those sequences that came from cultured and uncultured organisms. Overall, `r b_percent_cult_nseqs_all`% of bacterial sequences and `r a_percent_cult_nseqs_all`% of archaeal sequences have come from isolated organisms. Comparing the fraction of sequences deposited during and before 2006 from isolates to those collected after 2006, we found that culturing rates lag by `r  b_shift_cult_nseqs_all` and `r a_shift_cult_nseqs_all`-fold for bacteria and archaea, respectively. Among the `r nrow(b_cult_time)` bacterial phyla, `r sum(b_cult_time$all_cult_nseqs == 0)` have no cultured representatives and `r nrow(a_cult_time)` of the `r sum(a_cult_time$all_cult_nseqs == 0)` archaeal phyla have cultured representatives. This lag is likely due to the differences in throughput of culture-dependent and -independent approaches. Of the phyla with at least one cultured representative, the median percentage of sequences coming from a culture was only `r median(b_rate_cult_nseqs_phyulm[b_rate_cult_nseqs_phyulm!=0])*100`% for the bacterial phyla and `r median(a_rate_cult_nseqs_phyulm[a_rate_cult_nseqs_phyulm!=0])*100`% for the archaeal phyla (Figure 3). So, even though many phyla have cultured representatives, there is still a skew in the representation of most phyla found in cultivation efforts. Considering the possibility that large culture-independent sequencing efforts may only be re-sequencing organisms that already exist in culture, we asked what percentage of OTUs had at least one cultured representative. We found that `r b_percent_cult_otu_all`% of the `r sum(b_cult_time[,'all_total_sobs'])` bacterial OTUs and `r a_percent_cult_otu_all`% of the `r sum(a_cult_time[,'all_total_sobs'])` archeael OTUs had at least one cultured representative (Figure 5). Comparing the percentage of sequences with cultured representatives to the percentage of OTUs containing a sequence from a cultured representative revealed strong cultivation biases for several phyla associated with model/biomedically relevant organisms (Figure 5). The Tenericutes, Proteobacteria, Firmicutes, and Spirochaeatae all had considerably higher percentages of sequences generated by cultivated representatives than would be expected based on the number of cultured organisms represented by OTUs. This likely reflects the extremely high number of cultivated  *Mycobacterium tuberculosis*, *Escherichia coli*, *Bacillus*, *Streptococcus*, *Lactobacillus*, *Staphylococcus*, *Borrelia*, and others. Conversely, clades such as the *Actinobacteria*, *Fusobacteria*, and *Bacteroidetes* had a lower percentage of sequences belonging to cultivated representatives than would be expected based on the percentage of OTUs that have sequences from cultured organisms, indicating that the cultivation efforts in these clades are relatively efficient with regards to available diversity. Regardless of these observations, the majority of OTUs from any phylum remain uncultivated, to say nothing of the diversity of organisms that may be encapsulated within the 97% sequence identity cutoff.


```{r single_cell}
overlap_counts <- read.table(file="data/process/otu_overlap_by_method.tsv", header=T)
```


***New technologies to access novel biodiversity.***
Given the shift from Sanger sequencing to platforms that offer higher throughput but shorter reads, there is concern that our ability to harvest full-length sequences from communities will remain stalled. Several culture-independent methods have been developed that offer the ability to obtain full-length sequences of the 16S rRNA gene and even the complete genome. These have included single cell genomics and assembly of short 16S rRNA gene fragments using data generated from PCR amplicons or metagenomic shotgun sequence data using the Expectation-Maximization Iterative Reconstruction of Genes from the Environment (EMIRGE) algorithm [@refs]. To test the ability of these technologies to expand our knowledge of microbial diversity beyond that of traditional culutre-independent amplification and cultivation-based approaches, we compared the overlap of OTUs found using each of the new methods with the traditional approaches (Figure 6). Utilizing the 16S rRNA gene sequences extracted from the single-cell genomes available on the Integrated Microbial Genomes (IMG) system [@REF], we identified `r overlap_counts[overlap_counts$domain=="bacteria" & overlap_counts$method=="sc", 'n_seqs']` bacterial and `r overlap_counts[overlap_counts$domain=="archaea" & overlap_counts$method=="sc", 'n_seqs']` archaeal sequences that met our criteria, which were assigned to `r overlap_counts[overlap_counts$domain=="bacteria" & overlap_counts$method=="sc", 'n_otus']` and `r overlap_counts[overlap_counts$domain=="archaea" & overlap_counts$method=="sc", 'n_otus']` bacterial and archaeal OTUs, respectively. Interestingly, only `r overlap_counts[overlap_counts$domain=="bacteria" & overlap_counts$method=="sc", 'sc']` and `r overlap_counts[overlap_counts$domain=="archaea" & overlap_counts$method=="sc", 'sc']`% of the bacterial and archaeal single celled OTUs, respectively had not been observed by cultivation or PCR-based efforts. Next, we identified three studies that utilized EMIRGE to assemble 16S rRNA gene sequences from metagenomic sequences. Together these studies assembled `r overlap_counts[overlap_counts$domain=="bacteria" & overlap_counts$method=="em_meta", 'n_seqs']` bacterial and `r overlap_counts[overlap_counts$domain=="archaea" & overlap_counts$method=="em_meta", 'n_seqs']` archaeal full-length sequences, which were assigned to `r overlap_counts[overlap_counts$domain=="bacteria" & overlap_counts$method=="em_meta", 'n_otus']` and `r overlap_counts[overlap_counts$domain=="archaea" & overlap_counts$method=="em_meta", 'n_otus']` bacterial and archaeal OTUs, respectively. Only
`r overlap_counts[overlap_counts$domain=="bacteria" & overlap_counts$method=="em_meta", 'cultured']` and `r overlap_counts[overlap_counts$domain=="bacteria" & overlap_counts$method=="em_meta", 'pcr']`% of the bacterial OTUs generated by this approach were previously identified by this traditional cultivation and PCR-based approaches, respectively. Although the application of this approach to Archaea has been limited, it was still suprising that `r overlap_counts[overlap_counts$domain=="archaea" & overlap_counts$method=="em_meta", 'cultured']` and `r overlap_counts[overlap_counts$domain=="archaea" & overlap_counts$method=="em_meta", 'pcr']`% of the archaeal OTUs had been previously recovered by traditional cultivation and PCR-based approaches, respectively. Finally, we
pooled `r overlap_counts[overlap_counts$domain=="bacteria" & overlap_counts$method=="em_pcr", 'n_seqs']` bacterial sequences from three studies that utilized EMIRGE to assemble 16S rRNA gene sequences from fragmented amplicons [@REF]. These sequences were assigned to `r overlap_counts[overlap_counts$domain=="bacteria" & overlap_counts$method=="em_pcr", 'n_otus']` OTUs. We were surprised that only  `r 100-overlap_counts[overlap_counts$domain=="bacteria" & overlap_counts$method=="em_pcr", 'em_pcr']`% of these OTUs were previously found by a more traditional approach. Although these PCR-based EMIRGE results may be valid, the high degree of novelty that was observed suggests that the error of the assembled reads may be too high for generating reference sequences. Each of these methods represent promising opportunities to continue the bacterial census using full-length sequences as well as genomic information.


## Conclusions

In spite of estimates suggesting the global bacterial species richness ranges between 10^XX^ and 10^XX^ species, the current census based on full-length 16S rRNA gene sequences suggests that our current methods and sampling schemes would never reach these estimates. As we have shown, current strategies repeatedly sample the same OTUs and do a poor job of resampling rarer populations. Given this low level of OTU coverage, it is likely that there are many more bacterial and archaeal populations yet to be sampled. The recent advances in sequencing technologies have done little to expand the deposition of full-length sequences representing microbial biodiversity. We found that most sequences deposited into public databases are being made by a small number of projects that have deeply sampled similar environments. Furthermore, the number of full-lenth reads deposited into the databases has stalled. We fear that these trend will worsen unless we can leverage new sequencing and cultivations technologies to generate large numbers of full-length sequences.

During the period prior to the introduction of massively parallelized high throughput sequencing, it was common for a study to to generate dozens or hundreds of sequences per sample. The existing databases that are used for classifying sequences are based on these sequences. Thus, the sequences from those studies represent organisms that are generally abundant. We hypothesize that recent difficulties obtaining adequate classification for short sequences captured from more rare organisms is because our databases do not contain full-length references for those sequences. As stated in the 2004 census, it remains true that we need to expand the diversity of environments that are deeply sequenced to improve OTU coverage. Efforts to census microbial life using short read technology such as the International Census of Marine Microbes, the Earth Microbiome Project, and the Human Microbiome Project have significantly advanced our knowledge of microbial biogeography; however, these analyses have demonstrated the limitations of databases and taxonomies that are based on sequences from common and abundant organisms.

It is clear that considerable biodiversity has been discovered since the first census in 2004. However, much of it has been biased towards particular phyla and environments. Our analysis suggests that `r b_goods`% of new full-length bacterial and archaeal sequences are likely to have already been seen. Meanwhile, `r b_schloss`% of bacterial and `r a_schloss`% of archaeal OTUs have only been observed once. It appears that although brute force deep sequencing of communities may increase our coverage of OTUs, new methods are needed to more directly sample rare organisms. Nevertheless, novel technologies such as single-cell genomics, metagenomics, and algorithms to recover full-length sequences from new sequencing platforms have demonstrated promise in circumventing previous limitations in identifying new OTUs. Using EMIRGE to assemble fragmented 16S rRNA gene amplicons may allow us to obtain deep coverage of communities; however, it is still unclear how faithful the assembled sequence is to that of the original organism. Additional sequencing technologies also offer the ability to directly generate full-length sequences, such as PacBio and potentially Oxford Nanopore. Initial application of PacBio to sequencing full-length fragments suggests that the sequences suffer from a high error rate. To obtain a more direct investigation of rare organisms, microbiologists are developing novel cultivation and single cell genomics techniques. The ability to enrich or select for specific populations using these approaches could limit the need for redundant brute force sequencing. One strength of cultivation, single cell genomics, and metagenomics approaches is that not only do the methods yield a full 16S rRNA gene sequence, but they also allow researchers to obtain the complete genome of the organism. These approaches are still in active development, we hope that through continuous refinement, they may allow us to significantly improve the coverage of OTUs in public databases.

Our ability to assess the microbial census is limited by the quality of the data in the databases. First, it is widely acknowledged that the primers used to amplify 16S rRNA genes are biased; these biases are amplified when designing primers to amplify subregions used in sequencing short reads (Furhman, 2015). Furthermore, assembly of metagenomic data has shown the presence of introns in the 16S rRNA genes of organisms within the so-called "Candidate Phyla Radiation” (e.g. Saccharibacteria (TM7), Peregrinibacteria, Berkelbacteria (ACD58), WWE3 Microgenomates (OP11), Parcubacteria (OD1), et al.), that would preclude detection with standard PCR-based approaches [@Brown et al. 2015]. Thus, for many environments, the estimates for the census must be considered as lower bounds. Although the primers used to amplify the entire gene are also biased, obtaining a better characterization of microbial diversity will help improve the design of primers targeting subregions of the gene. Second, the willingness of researchers to contribute their sequences and the metadata describing the environment that the sequences were sampled from is critical for assessing the progress of the census and to accrue the benefits from having full-length sequences in the databases. Interestingly, the first 16S sequence was published in 1978, but was not available in a database until 1983. Similarly, most of the studies that used EMIRGE provided their sequences in supplements, not GenBank making them effectively invisible to 16S rRNA gene-specific databases. As assembly and long read technologies advance, a mechanism is needed to assess the quality of the consensus sequences and to make them easily accessible to the 16S rRNA gene-specific databases.

The advance in sequencing technologies has significantly expanded our ability to peer into the depths of microbial communities. We now have unprecedented abilities to sequence large numbers of individual communities deeply for the same cost of generating hundreds of full length sequences. Yet, unless we invest in improving our full-length databases, it will continue to be difficult to contextualize this newly discovered biodiversity within the tree of life.


## Materials and Methods

***Sequence data curation.***
The July 19, 2015 release of the ARB-formatted SILVA small subunit (SSU) reference database (SSU Ref v.123) was downloaded from http://www.arb-silva.de/fileadmin/silva_databases/release_123/ARB_files/SSURef_123_SILVA_19_07_15_opt.arb.tgz [@REF]. This release is based on the EMBL-EBI/ENA Release 123, which was released in March 2015. The SILVA curators identify potential SSU sequences using keyword searches and sequence-based search using RNAmmer (http://www.arb-silva.de/documentation/release-123/). The SILVA curators then screened the 7,168,241 resulting sequences based on a minimum length criteria (<300 nt), number of ambiguous base calls (>2%), length of sequence homopolymers (>2%), presence of vector contamination (>2%), low alignment quality value (<75), and likelihood of being chimeric (Pintail value < 50). Of the remaining sequences, the bacterial reference set retained those bacterial sequences longer than 1,200 nt and the archaeal reference set retained those archaeal sequences longer than 900 nt. The aligned 1,515,024 bacterial and 59,240 archaeal sequences were exported from the database using ARB along with the complete set of metadata. Additional sequence data was included from single-cell genomes available on the Integrated Microbial Genomes (IMG) system [@REF], many of which were recently obtained via the GEBA-MDM effort in Rinke et al. [@REF]. "SCGC” was searched on the IMG database March 12, 2015 to download the bacterial (N=249) and archaeal (N=46) 16S rRNA gene sequences and their associated metadata. Further, sequences generated from amplicon and shotgun metagenomic data using the EMIRGE program were also included. These were obtained from XXXXXXXXXX. The IMG and EMIRGE sequences were aligned against the respective SILVA-based reference using mothur [@REF]. The aligned bacterial and archaeal sequence sets were pooled and processed in parallel in mothur. Using mothur, sequences were further screened to remove any sequence with more than 2 ambiguous base calls and trimmed to overlap the same alignment coordinates. The sequences in the resulting bacterial dataset overlapped bases 113 through 1344 of an *E. coli* reference sequence (V00348) and had a median length of 1,227 nt. The sequences in the resulting archaeal dataset overlapped positions 362 to 937 of a *Sulfolobus solfataricus* reference sequence (X03235) and had a median length of 580 nt. The archaeal sequences were considerably shorter than their initial length because it was necessary to find a common overlapping region across the sequences. The final datasets contained `r b_nseqs` bacterial and `r a_nseqs` archaeal 16S rRNA gene sequences. Sequences were assigned to OTUs using the average neighbor clustering algorithm [@REF].


***Metadata curation.***
The metadata that was contained within the SSU Ref database was used to expand our analysis beyond a basic count of sequences and the number of OTUs in each domain. The environmental origins of the 16S rRNA gene sequences were manually classified using seven broad "coarse" categories, and further refined to facilitate additional analyses with twenty-six more specific "fine" categories (Table S1). These were assigned based on manual curation of the "isolation_source" category within the ARB database associated with each of the sequences (Table 1). For source definitions that were not identifiable by online searches, educated guesses were made or they were placed into the coarse "Other” category. There were `r coverage_by_category_and_time["no_source", "bact_count"]` bacterial and `r coverage_by_category_and_time["no_source", "arch_count"]` archaeal sequences where an "isolation_source" term was not collected. We ascertained whether a sequence came from a cultured organism by including those sequences that had data in their "strain" or "isolate" fields within the database and excluded any sequences that had "Unc" as part of their database name as this is a convention in the database that represents sequences from uncultured organisms. Complete tables containing the ARB-provided metadata, taxonomic information, OTU assignment, and our environmental categorizations are available at [@figshare].


***Calculating coverage.***
Sequencing coverage (C~Sequence~) was quantified by two methods. The first was to use Good's coverage according to

$$C_{Sequence} = 1 - \frac{n_1}{N_t}$$

where n~1~ is the number of OTUs represented by only one sequence and N~t~ is the total number of sequences [@ref]. Although Good's coverage provides information about the success of the sequencing effort in sampling the most abundant organisms in a community, it does not directly provide information about the success of the sequencing effort in recovering previously unobserved OTUs. To quantify the ability of sequencing to identifying novel OTUs or, in other words, to quantify the "distance" in the peak of the rarefaction curves to their hypothetical asymptote, we defined "OTU coverage" (C~OTU~) as

$$C_{OTU} = 1 - \frac{n_1}{S_t}$$

where S~t~ is the total number of OTUs. Whereas Good’s coverage estimates the probability that a new sequence will have already been seen, OTU coverage estimates the probability that a new OTU will match an existing one. It is therefore an extension of Good’s coverage in that it quantifies the probability that, for any given set of sequences clustered into an OTU, that OTU will have already been seen. Thus, high Good’s coverage means that any new sequence is unlikely to be novel, and high OTU coverage means that any new OTU is unlikely to be novel.


***Data analysis.***
Our analysis made use of ARB (OS X v.6.0) [@Pruesse2007], mothur (v.1.37.0) [@Schloss2009], and R (v.`r paste(version$major, version$minor, sep='.')`) [@language2015]. Within R we utilized knitr (v.`r packageDescription("knitr")$Version`) [@xie2013dynamic] and openxlsx (v. `r packageDescription("openxlsx")$Version`) [@Walker2015] packages. A reproducible version of this manuscript including data extraction and processing is available at https://www.github.com/SchlossLab/Schloss_Census2_mBio_2016.


\newpage
\doublespacing

**Figure 1. Number of OTUs sampled among bacterial and archaeal 16S rRNA gene sequences for different OTU definitions and level of sequencing effort.** Rarefaction curves for different OTU definitions of Bacteria (A) and Archaea (B). Rarefaction curves for the coarse environments in Table 1 for Bacteria (C) and Archaea (D). The number of bacterial and archaeal OTUs observed among the longest sequences in the SILVA database continues to grow at a rate too slow to ever reach estimates of 10^6^ to 10^11^ bacterial species.


**Figure 2. Progression of the microbial census since the first full-length 16S rRNA gene sequence was deposited into GenBank in 1983.*** The number of bacterial and archaeal 16S rRNA gene sequences deposited (A) and the new OTUs they represent (B) has increased exponentially until the last several years when the rate of change has plateaued. For both bacterial and archaeal sequences, the number of studies that are responsible for depositing more than 50% of the sequences each year has been relatively small (C).


**Figure 3. Relative rate of sequence deposition for each bacterial and archaeal phylum before and after 2006 relative to the sequencing of all bacteria.** The figure shows the relative rates for those phyla with at least 1,000 sequences and the x-axis is on a log2 scale. The data for all bacterial and archaeal phyla are available in Supplemental Tables 2 and 3, respectively.


**Figure 4. Heatmap depicting the relative abundance of the most common bacterial and archaeal phyla across different environments.** Each environmental category exhibited a phylum-level signature although the bacterial census was dominated by sequences from the Firmicutes, Proteobacteria, Actinobacteria, and Bacteroidetes and the archaeal census was dominated by sequences from the Euryarchaeota and Thaumarchaeota. The ten most abundant phyla across all environmental categories are shown. The data for all bacterial and archaeal phyla are available in Supplemental Tables 4 and 5, respectively.


**Figure 5. The rate that sequences and OTUs are generated from bacterial and archaeal cultures relative to all sequences and OTUs by phlum.** Phyla with greater than 1,000 sequences are listed by domain. Open circles indicate the percentage of sequences in the database that match cultured organisms. Closed circles indicate the percentage of OTUs in this analysis that contain sequences belonging to a cultured organism. The data for all bacterial and archaeal phyla are available in Supplemental Tables 6 and 7, respectively.


**Figure 6. The percentage of bacterial and archaeal OTUs found by single cell genomics and EMIRGE using PCR or metagenomics that were also detected by other.** The bars comparing a method to itself indicate the percentage of OTUs that were only detected by that method.


\newpage


## References
