---
title: "**The status of the microbial census: an update**"
bibliography: references.bib
output:
  pdf_document:
    includes:
      in_header: header.tex
csl: mbio.csl
fontsize: 11pt
geometry: margin=1.0in
---


```{r knitr_settings, eval=TRUE, echo=FALSE, cache=FALSE}
opts_chunk$set("tidy" = TRUE)
opts_chunk$set("echo" = FALSE)
opts_chunk$set("eval" = TRUE)
opts_chunk$set("warning" = FALSE)
opts_chunk$set("cache" = FALSE)

inline_hook <- function(x){
	print(x)
	if(is.numeric(x)){
		if(abs(x - round(x)) < .Machine$double.eps^0.5){
			paste(format(x,big.mark=',', digits=0, scientific=FALSE))
		} else {
			paste(format(x,big.mark=',', digits=1, nsmall=1, scientific=FALSE))
		}

	} else {
    	paste(x)      
	}
}
knitr::knit_hooks$set(inline=inline_hook)


#bact <- read.table(file='data/process/bacteria.v123.metadata', header=T, row.names=1, stringsAsFactors=FALSE)
bact_year_deposited <- gsub("^(\\d\\d\\d\\d)-.*", "\\1", bact$date)

#arch <- read.table(file='data/process/archaea.v123.metadata', header=T, row.names=1, stringsAsFactors=FALSE)
#arch_year_deposited <- gsub("^(\\d\\d\\d\\d)-.*", "\\1", arch$date)
```

\vspace{35mm}


Patrick D. Schloss^1$\dagger$^, Rene Girard^2^, Thomas Martin^2^, Joshua Edwards^2^, and J. Cameron Thrash^2$\dagger$^

\vspace{40mm}

$\dagger$ To whom correspondence should be addressed: pschloss@umich.edu and thrashc@lsu.edu

1\. Department of Microbiology and Immunology, University of Michigan, Ann Arbor, MI 48109

2\. Department of Biological Sciences, Louisiana State University, Baton Rouge, LA 70803


\newpage
\linenumbers


## Abstract

Abstract goes here.


\newpage

## Importance

Importance goes here.



\newpage

## Introduction
In 1983, the full-length 16S rRNA gene sequence of *Escherichia coli* (accession J01695) was deposited into NCBI's GenBank making it the first of more than 10 million 16S rRNA gene sequences to be deposited into the database [@Brosius1978]. GenBank accessions represent nearly one-third of all sequences deposited in the database making it the best-represented gene. As Sanger sequencing has given way to to-called "next generation sequencing" technologies, hundreds of millions of 16S rRNA gene sequences have been deposited into the NCBI's Sequence Read Archive. The expansion in sequencing throughput and increased access to sequencing technology has allowed for more environments to be sequenced at a deeper coverage resulting in the identification of novel taxa. The ability to obtain sequence data from microorganisms without cultivation has radically altered our perspective of their role in nearly every environment from deep ocean sediment cores (e.g. accession AY436526) to the International Space Station (e.g. accession DQ497748). The effort to quantify the number of different organisms in a system remains fundamental to understanding ecology [@reference]. At the scale of microorganisms, small physical sizes, morphological ambiguity, and highly variable population sizes complicate this process. Furthermore, creating standards for delimiting what makes one microbe "different” from another has been contentious [@reference]. In spite of these challenges, we continue to peel back the curtain on the microbial world with the aid of more and more informative, if still limited, technologies like cultivation, 16S rRNA gene surveys, single cell technologies, and metagenomics.

The increase in sequencing volume has come at the cost of sequence length. The commonly used MiSeq-based sequencing platform from Illumina is widely used to sequence the approximately 250 bp V4 hypervariable region of the 16S rRNA gene; other schemes have used different parts of the gene that are generally shorter than 500 bp. Perhaps most disconcerting about this development is the sense that the increased read depth is being gained using short read platforms rather than the full-length sequences. Because these short reads are used for classification to existing taxa, we are missing the opportunity to propose novel candidate taxa and vastly underappreciating the biodiversity of microbial life. We likely lack the references necessary to adequately classify the novel biodiversity we are sampling when we generate 100-times the sequence data from a community than we did using full-length sequencing.

Previously, Schloss and Handelsman [-@Schloss2004] assigned the then available 56,215 partial rRNA gene sequences to operational taxonomic units (OTUs) that were available in the Ribosomal Database Project and concluded that the sampling methods of the time were insufficient to identify the previously estimated 10^7^ to 10^9^ different species [@Dykhuizen1998, @Curtis2002]. That census called for a broader and deeper characterization of all environments. Refreshingly, this challenge was largely met. There have been major investments in studying the Earth's microbiome using 16S rRNA gene sequencing through initiatives such as the Human Microbiome Project [@reference], the Earth Microbiome Project [@reference], and the International Census of Marine Microorganisms. But most important, the original census was performed on the cusp of radical developments in sequencing technologies. That advancement has largely moved the bulk of sequencing throughput from large sequencing centers to individual investigators and leveraged their diversity to expand the representation of organisms and environments represented in public databases.

Here we update the status of the microbial census with nearly or completely full-length 16S rRNA gene sequences. In the 13 years since the collection of data for Schloss and Handelsman’s analysis, the number of full-length sequences has grown exponentially, despite the overwhelming contemporary focus by most researchers on short-read technologies. This update to the census allows us to evaluate the relative sampling thoroughness for different environments and clades, and make an argument for the continued need to collection full-length sequence data from many systems that have a long history of study. Although there has been a robust growth in the number of full-length sequences deposited to GenBank since its creation in 1983, the rate of growth has stalled over the past 5 years and the deposits have been dominated by a handful of research groups studying a limited number of environments. As researchers consider coalescing into a Unified Microbiome Initiative [@Alivisatos2015], it will be important to balance the need for mechanism-based studies with the need to generate full-length reference sequences from a diversity of environments. In this, continued technological advances, such as the reconstruction of nearly full-length sequences from metagenomics with EMIRGE (Miller 2011) and inovations in culturing (references needed: iChip, Epstein dormancy, Thrash 2015; Gifford 2014), will be vital.


## Materials and Methods

```{r overview}
category_summary <- read.table(file="data/process/coverage_by_category_and_time.tsv", header=T)

b_nseqs <- category_summary["total", "bact_count"]
b_goods <- category_summary["total", "bact_good_coverage"]
b_schloss <- category_summary["total", "bact_schloss_coverage"]

a_nseqs <- category_summary["total", "arch_count"]
a_goods <- category_summary["total", "arch_good_coverage"]
a_schloss <- category_summary["total", "arch_schloss_coverage"]
```

***Sequence data curation.***
The July 19, 2015 release of the ARB-formatted SILVA small subunit (SSU) reference database (SSU Ref v.123) was downloaded from http://www.arb-silva.de/fileadmin/silva_databases/release_123/ARB_files/SSURef_123_SILVA_19_07_15_opt.arb.tgz [@REF]. This release is based on the EMBL-EBI/ENA Release 123, which was released in March 2015. The SILVA curators identify potential SSU sequences using keyword searches and sequence-based search using RNAmmer (http://www.arb-silva.de/documentation/release-123/). The SILVA curators then screened the 7,168,241 resulting sequences based on a minimum length criteria (<300 nt), number of ambiguous base calls (>2%), length of sequence homopolymers (>2%), presence of vector contamination (>2%), low alignment quality value (<75), and likelihood of being chimeric (Pintail value < 50). Of the remaining sequences, the bacterial reference set retained those bacterial sequences longer than 1,200 nt and the archaeal reference set retained those archaeal sequences longer than 900 nt. The aligned 1,515,024 bacterial and 59,240 archaeal sequences were exported from the database using ARB along with the complete set of metadata. Additional sequence data was included from single-cell genomes available on the Integrated Microbial Genomes (IMG) system [@REF], many of which were recently obtained via the GEBA-MDM effort in Rinke et al. [@REF]. "SCGC” was searched on the IMG database March 12, 2015 to download the bacterial (N=249) and archaeal (N=46) 16S rRNA gene sequences and their associated metadata. The IMG sequences were aligned against the respective SILVA-based reference using mothur [@REF]. The aligned bacterial and archaeal sequence sets from SILVA and IMG were pooled and processed in parallel in mothur. Using mothur, sequences were further screened to remove any sequence with more than 2 ambiguous base calls and trimmed to overlap the same alignment coordinates. The sequences in the resulting bacterial dataset overlapped bases 113 through 1344 of an *E. coli* reference sequence (V00348) and had a median length of 1,227 nt. The sequences in the resulting archaeal dataset overlapped positions 362 to 937 of a *Sulfolobus solfataricus* reference sequence (X03235) and had a median length of 580 nt. The archaeal sequences were considerably shorter than their initial length because it was necessary to find a common overlapping region across the sequences. The final datasets contained `r b_nseqs` bacterial and `r a_nseqs` archaeal 16S rRNA gene sequences. Sequences were assigned to OTUs using the average neighbor clustering algorithm [@REF].


***Metadata curation.***
The metadata that was contained within the SSU Ref database was used to expand our analysis beyond a basic count of sequences and the number of OTUs in each domain. The environmental origins of the 16S rRNA gene sequences were manually classified using seven broad "coarse" categories, and further refined to facilitate additional analyses with twenty-six more specific "fine" categories (Table 1). These were assigned based on manual curation of the "isolation_source" category within the ARB database associated with each of the sequences (Table 1). For source definitions that were identifiable by online searches, educated guesses were made or they were placed into the coarse "Other” category. There were `r category_summary["no_source", "bact_count"]` bacterial and `r category_summary["no_source", "arch_count"]` archaeal sequences where an "isolation_source" term was not collected. We ascertained whether a sequence came from a cultured organism by including those sequences that had data in their "strain" or "isolate" fields within the database and excluded any sequences that had "Unc" as part of their database name as this is a convention in the database that represents sequences from uncultured organisms. Complete tables containing the ARB-provided metadata, taxonomic information, OTU assignment, and our environemtnal categorizations are available at [@figshare].


***Data analysis.***
Our analysis made use of ARB (OS X v.6.0) [@Pruesse2007], mothur (v.1.37.0) [@Schloss2009], and R (v.`r paste(version$major, version$minor, sep='.')`) [@language2015]. Within R we utilized knitr (v.`r packageDescription("knitr")$Version`) [@xie2013dynamic] and openxlsx (v. `r packageDescription("openxlsx")$Version`) [@Walker2015] packages. A reproducible version of this manuscript including data extraction and processing is available at https://www.github.com/SchlossLab/Schloss_Census2_mBio_2015.


## Results and Discussion

```{r rarefaction}
# take these data out of the rarefaciton files so that we can have the unique,
# 0.03, 0.05, 0.10, and 0.20 OTU definitions for both the bacterial and archaeal
# data

get_sobs <- function(data){
	rownames(data) <- data[,1]
	data <- data[,-1]
	data <- data[,!grepl('ci', colnames(data))]
	unlist(data[nrow(data),])
}

b_sobs <- get_sobs(read.table(file='data/mothur/all_bacteria.filter.unique.precluster.an.rarefaction', header=T))
a_sobs <- get_sobs(read.table(file='data/mothur/all_archaea.filter.unique.precluster.an.rarefaction', header=T))
```

***The status of the bacterial and archaeal census.***
To assess the field's progress in characterizing the biodiversity of bacteria and archaea we assigned each 16S rRNA gene sequence to OTUs using distance threshold varying between 0 and 20%. Although it is not possible to link a specific taxonomic level (e.g. species, genus, family, etc.) to a specific distance threshold, we selected distances of 0, 3, 5, 10, and 20% because they are widely regarded as representing the range of genetic diversity of the 16S rRNA gene within each domain. By rarefaction, it was clear that the ongoing sampling efforts have started to saturate the number of current OTUs. After sampling `r b_nseqs` near full-length bacterial 16S rRNA gene sequences we have identified `r b_sobs['unique']`, `r b_sobs['X0.03']`, `r b_sobs['X0.05']`, `r b_sobs['X0.10']`, and `r b_sobs['X0.20']` OTUs at the selected thresholds (Figure 1A, Table 1). Using on the OTUs generated using a 3% threshold, we found that `r b_goods`% of the sequences belonged to OTUs that had been observed more than once. In contrast, only `r b_schloss`% of the OTUs that were observed had been seen more than once. Paralleling the bacterial results, after sampling `r nrow(arch)` archaeal 16S rRNA gene sequences we have identified `r a_sobs['unique']`, `r a_sobs['X0.03']`, `r a_sobs['X0.05']`, `r a_sobs['X0.10']`, and `r a_sobs['X0.20']` OTUs (Figure 1B, Table 1). Using on the OTUs generated using a 3% threshold, we found that `r a_goods`% of the sequences belonged to OTUs that had been observed more than once. In contrast, only `r a_schloss`% of the OTUs that were observed had been seen more than once. The results for the bacterial and archaeal censuses indicate that regardless of the domain, if we continue sampling with the current strategies we will continue to sample OTUs that have already been observed even though a large fraction of OTUs have only been sampled once. Furthermore, considering more than `r 100-min(b_schloss, a_schloss)`% of the OTUs have only been observed once it is likely that an even larger number of OTUs have yet to be sampled for both domains.



```{r time_course}
time_data <- read.table(file="data/process/by_year_analysis.tsv", header=TRUE, stringsAsFactors=FALSE)

all_nseqs_2009_2012 <- sum(time_data[as.character(2009:2012), 'b_nseqs'])
dominant_nseqs_2009_2012 <- sum(time_data[as.character(2009:2012), 'b_n50_nseqs'])

# what were the papers that these sequences came from?
dominant_pmids_2009_2012 <- unique(unlist(strsplit(time_data[as.character(2009:2012), "b_n50_doi"], split=',')))

#require(knitcitations, quiet=TRUE)
#citep(c('10.1038/ismej.2012.79', '10.1371/journal.pone.0026284', '10.1101/gr.131029.111', '10.1073/pnas.1004204107', '10.1126/science.1171700'))

stopifnot(dominant_pmids_2009_2012 == c("19478181", "22310478", "20668241", "22719818", "22832344"))
#if this stops then will need to update stuff below
```

***The status of the bacterial census.***
One explanation for the large number of OTUs that have only been observed once is that with the the broad adoption of highly parallelized sequencing platforms that generate short sequence reads, the rate of full-length sequence generation has declined. In fact, since 2009 the number of new bacterial sequences generated has slowed two an average of `r floor(mean(time_data[as.character(2009:2014),'b_nseqs']))` sequences per year (Figure 2A). Although this is still an impressive number of sequences, since `r start<-2007;start` the number of new bacterial OTUs has plateaued at an average of `r mean(time_data[as.character(start:2014),'b_sobs'])` new OTUs per year (Figure 2B). Given the expense of generating full-length sequences using the Sanger sequencing technology and the transition to other platforms at that time, we expected that the large number of sequences were being deposited by a handful of large projects. Indeed, when we counted the number of submissions responsible for depositing 50% of the sequences, we found that with the exception of 2006 and 2013, eight or fewer studies were responsible for depositing the majority of the full-length sequences each year since 2005 (Figure 2C). Between 2009 and 2012, `r all_nseqs_2009_2012` total sequences were submitted and `r sum(time_data[as.character(2009:2012), 'b_n50'])` submissions from `r length(unique(dominant_titles))` studies were responsible for depositing `r dominant_nseqs_2009_2012` (`r 100*dominant_nseqs_2009_2012/all_nseqs_2009_2012`% of all sequences). These studies generated sequences from the human gastrointestinal tract [@Li_2012], human skin [@Kong_2012; @Grice_2009], murine skin [@Grice_2010], and hypersaline microbial mats [@Kirk_Harris_2012]. In contrast to recent years, between 1995 and 2006, an average of `r mean(time_data[as.character(1995:2006), 'b_n50'])` studies were responsible for submitting more than half of the sequences each year. Although these deep surveys represent significant contributions to our knowledge of bacterial biogeography, their small number and lack of environmental diversity is indicative of the broader problems in advancing the bacterial census.


***The status of the archaeal census.***
The depth of sequencing being done to advance the archaeal census has been `r floor(sum(time_data$b_nseqs)/sum(time_data$a_nseqs))`-times less than that of the bacterial census (Table 1). The annual number of sequences submitted has largely paralleled that of the bacterial census with a plateau starting in 2009 and an average of `r mean(time_data[as.character(2009:2014),'a_nseqs'])` sequences each year since then. The number of new archaeal OTUs represented by these sequences began to slow in 2005 with an average of `r mean(time_data[as.character(2005:2014),'a_sobs'])` new OTUs per year. With the exception of 2012 and 2014, the number of submissions responsible for more than 50% of the archaeal sequences submitted per year has varied between `r min(time_data[as.character(c(1996:2011,2013)),'a_n50'])` and `r max(time_data[as.character(c(1996:2011,2013)),'a_n50'])` submissions per year. The clear bias towards sequencing bacterial 16S rRNA genes has limited the ability to more fully characterize the biodiversity of the archaea.

```{r taxon_census}
oxford_comma <- function(ids){
	if(length(ids)==0){
		"ERROR"
	} else if(length(ids) == 1){
		ids
	} else if(length(ids) == 2){
		paste(ids, collapse=" and ")
	} else {
		paste(paste(ids[1:(length(ids)-1)], collapse=", "), ids[length(ids)], sep=", and ")
	}
}

b_phylum <- read.table(file="data/process/bacteria.phyla.counts.tsv", header=T)
b_phylum <- b_phylum[!rownames(b_phylum) %in% 'aquifer1',]
rownames(b_phylum) <- gsub("Candidate_division_", "", rownames(b_phylum))
rownames(b_phylum) <- gsub("_", " ", rownames(b_phylum))

b_new_phyla <- rownames(b_phylum)[b_phylum$pre == 0]
b_new_phyla_names <- oxford_comma(b_new_phyla_names)


a_phylum <- read.table(file="data/process/archaea.phyla.counts.tsv", header=T)
a_new_phyla <- rownames(a_phylum)[a_phylum$pre == 0]

a_new_phyla_names <- oxford_comma(a_new_phyla_names)
a_new_phyla_names <- gsub("\\(\\S*\\)", "", a_new_phyla_names)
a_new_phyla_names <- gsub("_", " ", a_new_phyla_names)
```

***The ability to sample microbial life is taxonomically skewed (meh.)***
The Firmicutes, Proteobacteria, Actinobacteria, and Bacteroidetes represent `r 100*sum(b_phylum[c("Firmicutes", "Proteobacteria", "Actinobacteria", "Bacteroidetes"), "all"])/sum(b_phylum$all)`% of the bacterial sequences and the Euryarchaeota and Thaumarchaeota `r 100*sum(a_phylum[c("Euryarchaeota",  "Thaumarchaeota"), "all"])/sum(a_phylum$all)`% of the archaeal sequences. We sought to understand how the representation of individual phyla has changed relative to the state of the census in 2006. We used 2006 as a reference point for calibrating the dynamics of the bacterial and archaeal censuses since that was the year that the first highly parallelized 16S rRNA gene sequence dataset was published and ushered in a radical change in how microbial communities are studied [@Sogin2006]. In 2006 there were `r sum(b_phylum$pre!=0)` bacterial and `r sum(a_phylum$pre!=0)` phyla. Since then there have been `r b_new_phyla` new  bacterial (`r b_new_phyla_names`) and `r a_new_phyla` new archaeal candidate phyla (`r a_new_phyla_names`). Relative to the overall sequencing trends before and after 2006, several phyla stand out for being over and underrepresented in sequence submissions (Figure phylum_effort.pdf). Among the bacterial phyla with at least 1,000 sequences, `r oxford_comma(sort(rownames(b_phylum[b_phylum$ratio_of_ratios >= 4.0 & b_phylum$all >=1000,])))` were sequenced 4-fold more often while `r oxford_comma(sort(rownames(b_phylum[b_phylum$ratio_of_ratios < 0.5 & b_phylum$all >=1000,])))` were sequenced 2-fold less often than would have been expected since 2006.
Among the archaeal phyla with at least 1,000 sequences, the Thaumarchaeota were sequenced `r a_phylum["Thaumarchaeota", "ratio_of_ratios"]`-fold more often and the Crenarchaeota were sequenced `r 1/a_phylum["Crenarchaeota", "ratio_of_ratios"]`-fold less often than expected. Together, these results demonstrate a change in the phylum-level lineages represented in the census from before and after 2006.



```{r environment}
strip <- function(x){	as.numeric(gsub(',', '', x))	}

coverage_by_category_and_time <- read.table(file="data/process/coverage_by_category_and_time.tsv", header=T, stringsAsFactors=FALSE)

time <- grepl('_pre', colnames(coverage_by_category_and_time)) | grepl('_post', colnames(coverage_by_category_and_time))
coverage_by_category <- coverage_by_category_and_time[,!time]

totals <- unlist(coverage_by_category[nrow(coverage_by_category),])
coverage_by_category <- coverage_by_category[-nrow(coverage_by_category),]

b_class_percent <- 100*sum(coverage_by_category[-nrow(coverage_by_category), "bact_count"])/sum(totals["bact_count"])

a_class_percent <- 100*sum(coverage_by_category[-nrow(coverage_by_category), "arch_count"])/sum(totals["arch_count"])


coarse_map <- c("A"=="aerosol", "AQ" = "aquatic", "B" = "built environment", "O" = "other", "P" = "plant", "S" = "soil", "Z" = "zoonotic")

coarse <- gsub("^(.)[A-Z]$", "\\1", rownames(coverage_by_category))
coarse <- gsub("^(..)[A-Z]+$", "\\1", coarse)

b_agg_count <- aggregate(coverage_by_category$bact_count, by=list(coarse_map[coarse]), sum)
b_text <- paste0(b_agg_count$Group.1, ' (N=', format(b_agg_count$x, big.mark=',', trim=TRUE), ')')[order(b_agg_count$x, decreasing=T)]

a_agg_count <- aggregate(coverage_by_category$arch_count, by=list(coarse_map[coarse]), sum)
a_text <- paste0(a_agg_count$Group.1, ' (N=', format(a_agg_count$x, big.mark=',', trim=TRUE), ')')[order(a_agg_count$x, decreasing=T)]


phylum_category <- read.table(file="data/process/phylum_category_counts.tsv")
bacteria <- phylum_category[phylum_category$domain == "bacteria", -1]

dominant_bacteria <- sort(table(apply(bacteria, 2, which.max)), decreasing=T)
names(dominant_bacteria) <- rownames(bacteria)[as.numeric(names(dominant_bacteria))]
dominant_bacteria_list <- oxford_comma(paste0(names(dominant_bacteria), ' (N=', dominant_bacteria, ')'))

archaea <- phylum_category[phylum_category$domain == "archaea", -1]
dominant_archaea <- sort(table(apply(archaea, 2, function(x){z<-which.max(x);if(length(z)==0){z<-NA};z})), decreasing=T)
names(dominant_archaea) <- rownames(archaea)[as.numeric(names(dominant_archaea))]
dominant_archaea_list <- oxford_comma(paste0(names(dominant_archaea), ' (N=', dominant_archaea, ')'))

```

***Focusing the census by environment***
We were able to assign `r b_class_percent` and `r a_class_percent`% of the sequences to one of seven broad environmental categories based on the metadata that accompanied the SILVA database. Across these broad categories there was wide variation in the number of sequences that have been sampled. Among the bacterial sequences the three best represented groups were from `r oxford_comma(b_text[1:3])` sources and among the archaeal sequences the three best represented groups were from `r oxford_comma(a_text[1:3])` sources. For both domains, soil samples were the fourth most represented category (bacteria: `r b_agg_count[b_agg_count$Group.1=="soil", "x"]`; archaea: `r a_agg_count[a_agg_count$Group.1=="soil", "x"]`). The orders of these categories was surprising considering soil and aquatic environments harbor the most microbial biomass and biodiversity [@Whitman1998]. In spite of wide variation in sequencing depth and coverage (Table 1), the interquartile range across the fine-level categories for the bacterial OTU-based coverage only varied between `r quantile(coverage_by_category$bact_schloss_coverage, prob=c(0.25))` to `r quantile(coverage_by_category$bact_schloss_coverage, prob=c(0.75))` (median coverage=`r median(coverage_by_category$bact_schloss_coverage, na.rm=T)`%). The interquartile range in the OTU-based coverage by environment for the archaeal data was `r quantile(coverage_by_category$arch_schloss_coverage, prob=c(0.25), na.rm=T)` to`r quantile(coverage_by_category$arch_schloss_coverage, prob=c(0.75), na.rm=T)` (median coverage=`r median(coverage_by_category$arch_schloss_coverage, na.rm=T)`%). The archaeal coverage was higher than that of the bacterial OTU coverage for all categories except the food-associated, plant surface, and other invertebrate categories. Across all categories, the bacterial and archaeal sequencing data represented a limited number of phyla (Figure category_phylum_heatmap.pdf). Among the bacterial data, the fine-scale categories were dominated by `r dominant_bacteria_list` and among the archaeal data, they were dominated by `r dominant_archaea_list`. Regardless, there were clear phylum-level signatures that differentiated the various categories. Within each of the bacterial and archaeal phyla, there was considerable variation in the relative abundance of each across the categories confirming that taxonomic signatures exist to differentiate different environments even at a broad taxonomic level.




```{r culture}
	b_is_cult <- (!is.na(bact$strain) | !is.na(bact$isolate)) & !grepl("\\.Unc", rownames(bact))
	a_is_cult <- (!is.na(arch$strain) | !is.na(arch$isolate)) & !grepl("\\.Unc", rownames(arch))

	b_ncult_full <- sum(b_is_cult)
	a_ncult_full <- sum(a_is_cult)

	b_fract_cult <- 100 * b_ncult_full / nrow(bact)
	a_fract_cult <- 100 * a_ncult_full / nrow(arch)


	b_ncult_2005 <- sum(b_is_cult & bact_year_deposited <= 2005, na.rm=T)
	a_ncult_2005 <- sum(a_is_cult & arch_year_deposited <= 2005, na.rm=T)


	b_ncult_delta <- b_ncult_full - b_ncult_2005

	b_before_2005 <- sum(bact_year_deposited <= 2005, na.rm=T)
	b_since_2005 <- sum(bact_year_deposited > 2005, na.rm=T)

	b_slow_cult <- (b_since_2005/b_before_2005) / (b_ncult_delta/b_ncult_2005)

	a_ncult_delta <- a_ncult_full - a_ncult_2005

	a_before_2005 <- sum(arch_year_deposited <= 2005, na.rm=T)
	a_since_2005 <- sum(arch_year_deposited > 2005, na.rm=T)

	a_slow_cult <- (a_since_2005/a_before_2005) / (a_ncult_delta/a_ncult_2005)


	bact_phylum <- gsub("^[^;]*;([^;]*);.*", "\\1", bact$taxonomy)
	bact_phylum[bact_phylum=="Euryarchaeota"] <- NA
	bact_phylum[bact_phylum=="aquifer1"] <- NA

	b_is_cult[rownames(bact) == 'AJ224540.MxeCultu'] <- FALSE
#	aquifer1 - scrap
#	bact[bact_phylum=="Candidate_division_OP3" & !is.na(bact$isolate) & !is.na(bact_phylum),] - AJ224540.MxeCultu

#	bact[bact_phylum=="CKC4" & !is.na(bact$strain) & !is.na(bact_phylum),] -> 2 - 1 is a genome, 1 is an isolate from same research group
#	bact[bact_phylum=="PAUC34f" & (!is.na(bact$isolate) | !is.na(bact$strain)),] - legit
#	bact[bact_phylum=="TM6" & (!is.na(bact$isolate) | !is.na(bact$strain)) & !grepl("\\.Unc", rownames(bact)),] - legit

	b_phylum_cult_table <- table(bact_phylum, b_is_cult)
	b_phylum_cult_rate <- b_phylum_cult_table[,"TRUE"] / apply(b_phylum_cult_table, 1, sum)


	arch_phylum <- gsub("^[^;]*;([^;]*);.*", "\\1", arch$taxonomy)
	arch_phylum[arch_phylum=="Archaea;"] <- "Unclassified"
	a_phylum_cult_table <- table(arch_phylum, a_is_cult)
	a_phylum_cult_rate <- a_phylum_cult_table[,"TRUE"] / apply(a_phylum_cult_table, 1, sum)


	b_cult_otus <- unique(bact$otu[b_is_cult])
	b_in_cult_otu <- bact$otu %in% b_cult_otus

	n_all_otus_phylum <- aggregate(bact$otu, by=list(bact_phylum), function(x){length(unique(x))})
	n_cult_otus_phylum <- aggregate(bact$otu[b_in_cult_otu], by=list(bact_phylum[b_in_cult_otu]), function(x){length(unique(x))})

	b_otus_phylum <- data.frame(matrix(rep(0, nrow(n_all_otus_phylum)*2), ncol=2, dimnames=list(n_all_otus_phylum$Group.1, c('cultured', 'total'))))

	b_otus_phylum[n_all_otus_phylum$Group.1,"total"] <- n_all_otus_phylum$x
	b_otus_phylum[n_cult_otus_phylum$Group.1,"cultured"] <- n_cult_otus_phylum$x


	a_cult_otus <- unique(arch$otu[a_is_cult])
	a_in_cult_otu <- arch$otu %in% a_cult_otus

	n_all_otus_phylum <- aggregate(arch$otu, by=list(arch_phylum), function(x){length(unique(x))})
	n_cult_otus_phylum <- aggregate(arch$otu[a_in_cult_otu], by=list(arch_phylum[a_in_cult_otu]), function(x){length(unique(x))})

	a_otus_phylum <- data.frame(matrix(rep(0, nrow(n_all_otus_phylum)*2), ncol=2, dimnames=list(n_all_otus_phylum$Group.1, c('cultured', 'total'))))

	a_otus_phylum[n_all_otus_phylum$Group.1,"total"] <- n_all_otus_phylum$x
	a_otus_phylum[n_cult_otus_phylum$Group.1,"cultured"] <- n_cult_otus_phylum$x

	#not sure what this is actually getting...
	b_n1_cult_otus <- 100*sum(table(bact$otu[b_in_cult_otu])==1)/length(table(bact$otu[b_in_cult_otu]))
	a_n1_cult_otus <- 100*sum(table(arch$otu[a_in_cult_otu])==1)/length(table(arch$otu[a_in_cult_otu]))
```

***The cultured census***
In the 2004 bacterial census there was great concern that although culture-independent methods were significantly enhancing our knowledge of microbial life, there were numerous bacterial phyla with no or only a few cultured representatives. To update this assessment, we identified those sequences that came from cultured and uncultured organisms. Overall, `r b_fract_cult`% of bacterial sequences and `r a_fract_cult`% of archaeal sequences have come from isolated organisms. Comparing the fraction of sequences deposited during and before 2006 from isolates to those collected after 2006, we found that culturing rates lag by `r  b_slow_cult` and `r a_slow_cult`-fold for bacteria and archaea, respectively. Among the `r length(b_phylum_cult_rate)` bacterial phyla, `r sum(b_phylum_cult_rate == 0)` have no cultured representatives and `r sum(b_phylum_cult_rate == 0)` of the `r length(a_phylum_cult_rate)` archaeal phyla have cultured representatives. Of the phyla with at least one cultured representative, the median percentage of sequences coming from a culture was only `r median(b_phylum_cult_rate[b_phylum_cult_rate!=0])*100`% for the bacterial phyla and `r median(a_phylum_cult_rate[a_phylum_cult_rate!=0])*100`% for the archaeal phyla (Figure XXXXXX). When we asked what percentage of OTUs had at least one cultured respresentative we found that `r 100*sum(b_otus_phylum$cultured)/sum(b_otus_phylum$total)`% of the `r sum(b_otus_phylum$total)` bacterial OTUs and `r 100*sum(a_otus_phylum$cultured)/sum(a_otus_phylum$total)`% of the `r sum(a_otus_phylum$total)` archeael OTUs had at least one cultured representative. Of those OTUs that had a sequence from a culture, `r b_n1_cult_otus`
and `r a_n1_cult_otus`% of them only contained sequences from cultures. These values suggest that bacterial cultivation efforts are biased towards

* need a plot showing % cultured sequences and % cultured OTUs
	* fraction of sequences from cultured / uncultured bugs (2014) by phylum
		- b_phylum_cult_rate
		- a_phylum_cult_rate
	* % of OTUs with a cultured representative by phylum and in aggregate
		- b_otus_phylum
		- a_otus_phylum



```{r single_cell}


grepl("^[^\\.]*$", rownames(bact))

dim(bact[bact$publication_doi=="10.1038/nature12352" & !is.na(bact$publication_doi),])
#[1] 95 31


```
***New technologies to access novel biodiversity***
Assembly of metagenomic and single cell shotgun sequence data offers the hope of identifying large fragments of genomic data from as yet uncultured organisms. To test the ability of single cell technologies to identify


  - Single cell genomics
  - Metagenomes




## Conclusions

Future for PacBio in generating full-length sequences

The first 16S sequence was published in 1978, not deposited until 1983. A bit of an allegory for our time.

Impact of EMIRGE
- http://www.genomebiology.com/2011/12/5/R44

Renewed call for cultivation

What are the most significant improvements since 2004, and where are we still lacking the most data?

Recent data suggests that a considerable diversity of microorganisms may be missing based on biases in existing 16S rRNA gene primers. This dataset does not include sequences from metagenomic assemblies but Brown et al. (2015) have used such assemblies to show evidence for introns in the 16S rRNA genes of organisms within the so-called "Candidate Phyla Radiation” (CPR- Saccharibacteria (TM7), Peregrinibacteria, Berkelbacteria (ACD58), WWE3 Microgenomates (OP11), Parcubacteria (OD1), et al.), that would preclude detection with standard cultivation-independent microbial surveys. Furthermore, many of these CPR organisms are very small and frequently pass through 0.2 µm filters (Luef, 2015). Thus, for many environments, the estimates within must be considered as lower bounds.

Generating a comprehensive understanding of any system with a single gene may seem a fool’s errand, yet we have learned a considerable amount regarding the diversity, dynamics, and natural history of microorganisms using the venerable 16S rRNA gene. Indeed, continual community efforts to obtain 16S rRNA gene assessments of every environment possible have presented us with an ever-increasing estimate of total microbial diversity and the concomitant excitement of frontier science. While reliance on this gene subjects us to biases created by primer selection [REFS], differences in amplification strength [REFS] and fidelity [REFS], internal features which may disrupt traditional measurements [REFS], and potentially misleading classification due to infrequent horizontal gene transfer [REFS], the total data available from persistent collection of 16S rRNA gene sequences nevertheless dwarfs that of any other genetic marker. Thus, an attempt to quantify how much of the microbial world has been revealed inevitably starts there.

Given the relatively low number of archaeal sequences that have been deposited for many of these categories, it is possible that the coverage for many categories may not be reliable (e.g. aerosol and plant surfaces). By decomposing the census by environmental categories, it is clear that even among the best sampled environments, our ability to claim anything but a basic characterization of microbial biodiversity is limited.


\newpage
\singlespacing



**Figure 1. Sequences deposited by year**

Pat, this is a placeholder for you

**Figure 2. Sequence deposition by study**

Pat, ibid.

**Figure 3. Sampling by OTU cutoff**

I only have an old code from Rene for this, and I don't think it reports all the sequences


**Figure 4A. Bacterial sampling by environment**

```{r figure4a, eval=FALSE}
data <- read.table(file="data/mothur/all_bacteria.env_category.groups.rarefaction", header=T)
category <- "\\.plant$" #change the SO to whatever category you want (leave the \\. and $)
plant_columns <- grep(category, colnames(data))
category <- "\\.aerosol$" #change the SO to whatever category you want (leave the \\. and $)
aerosol_columns <- grep(category, colnames(data))
category <- "\\.aquatic$" #change the SO to whatever category you want (leave the \\. and $)
aquatic_columns <- grep(category, colnames(data))
category <- "\\.built$" #change the SO to whatever category you want (leave the \\. and $)
built_columns <- grep(category, colnames(data))
category <- "\\.other$" #change the SO to whatever category you want (leave the \\. and $)
other_columns <- grep(category, colnames(data))
category <- "\\.soil$" #change the SO to whatever category you want (leave the \\. and $)
soil_columns <- grep(category, colnames(data))
category <- "\\.zoonotic$" #change the SO to whatever category you want (leave the \\. and $)
zoonotic_columns <- grep(category, colnames(data))
plant_aerosol_aquatic_built_other_soil_zoonotic_data <- data[,c(1, plant_columns, aerosol_columns, aquatic_columns, built_columns, other_columns, soil_columns, zoonotic_columns)]  #the 1 gets the first column which is numsampled
nrows_to_keep <- which.max(plant_aerosol_aquatic_built_other_soil_zoonotic_data$numsampled[!is.na(plant_aerosol_aquatic_built_other_soil_zoonotic_data[,2])])
plant_aerosol_aquatic_built_other_soil_zoonotic_data <- plant_aerosol_aquatic_built_other_soil_zoonotic_data[1:nrows_to_keep,]
pdf(file="/Users/renegirard7gfy/Desktop/Bac_Uniq_3per.pdf")

par(mar=c(5,5,0.5,0.5))
plot(NA, type="n", ylim=c(0,max(40000)),
     xlim=c(0,max(data$numsampled)),
     xlab="Number of sequences sampled",
     ylab="",
     axes=FALSE)

points(data$X0.03.plant~data$numsampled, type="l", col="red", lwd=2)
points(data$X0.03.aerosol~data$numsampled, type="l", col="green", lwd=2)
points(data$X0.03.aquatic~data$numsampled, type="l", col="blue", lwd=2)
points(data$X0.03.built~data$numsampled, type="l", col="orange", lwd=2)
points(data$X0.03.other~data$numsampled, type="l", col="black", lwd=2)
points(data$X0.03.soil~data$numsampled, type="l", col="purple", lwd=2)
points(data$X0.03.zoonotic~data$numsampled, type="l", col="deeppink", lwd=2)

legend(x=5e5, y=20000, lwd=2, lty=1,
       legend=c("Other", "Plant", "Aerosol", "Aquatic", "Built", "Soil", "Zoonotic"),
       col=c("black", "red", "green", "blue", "orange", "purple", "deeppink"))

axis(1, at=seq(0,700000,1e5), label=c("0", "100,000", "200,000", "300,000", "400,000", "500,000", "600,000", "700,000"), cex.axis=0.8)
axis(2, at=seq(0,40000,5000), label=c("0", "5,000", "10,000", "15,000", "20,000", "25,000", "30,000", "35,000", "40,000"), las=2, cex.axis=0.8)
box()
mtext(side=2, "Number of OTUs", line=4)

dev.off()

```

**Figure 4B. Archaeal sampling by environment**

```{r figure4b, eval=FALSE}

data <- read.table(file="data/mothur/all_archaea.env_category.groups.rarefaction", header=T)
category <- "\\.plant$" #change the SO to whatever category you want (leave the \\. and $)
plant_columns <- grep(category, colnames(data))
category <- "\\.aerosol$" #change the SO to whatever category you want (leave the \\. and $)
aerosol_columns <- grep(category, colnames(data))
category <- "\\.aquatic$" #change the SO to whatever category you want (leave the \\. and $)
aquatic_columns <- grep(category, colnames(data))
category <- "\\.built$" #change the SO to whatever category you want (leave the \\. and $)
built_columns <- grep(category, colnames(data))
category <- "\\.other$" #change the SO to whatever category you want (leave the \\. and $)
other_columns <- grep(category, colnames(data))
category <- "\\.soil$" #change the SO to whatever category you want (leave the \\. and $)
soil_columns <- grep(category, colnames(data))
category <- "\\.zoonotic$" #change the SO to whatever category you want (leave the \\. and $)
zoonotic_columns <- grep(category, colnames(data))
plant_aerosol_aquatic_built_other_soil_zoonotic_data <- data[,c(1, plant_columns, aerosol_columns, aquatic_columns, built_columns, other_columns, soil_columns, zoonotic_columns)]  #the 1 gets the first column which is numsampled
nrows_to_keep <- which.max(plant_aerosol_aquatic_built_other_soil_zoonotic_data$numsampled[!is.na(plant_aerosol_aquatic_built_other_soil_zoonotic_data[,2])])
plant_aerosol_aquatic_built_other_soil_zoonotic_data <- plant_aerosol_aquatic_built_other_soil_zoonotic_data[1:nrows_to_keep,]
pdf(file="/Users/renegirard7gfy/Desktop/Arc_Uniq_3per.pdf")

par(mar=c(5,5,0.5,0.5))
plot(NA, type="n", ylim=c(0,max(data[,-1], na.rm=T)),
     xlim=c(0,max(data$numsampled)),
     xlab="Number of sequences sampled",
     ylab="",
     axes=FALSE)

points(data$X0.03.plant~data$numsampled, type="l", col="red", lwd=2)
points(data$X0.03.aerosol~data$numsampled, type="l", col="green", lwd=2)
points(data$X0.03.aquatic~data$numsampled, type="l", col="blue", lwd=2)
points(data$X0.03.built~data$numsampled, type="l", col="orange", lwd=2)
points(data$X0.03.other~data$numsampled, type="l", col="black", lwd=2)
points(data$X0.03.soil~data$numsampled, type="l", col="purple", lwd=2)
points(data$X0.03.zoonotic~data$numsampled, type="l", col="deeppink", lwd=2)

legend(x=15000, y=1550, lwd=2, lty=1,
       legend=c("Other", "Plant", "Aerosol", "Aquatic", "Built", "Soil", "Zoonotic"),
       col=c("black", "red", "green", "blue", "orange", "purple", "deeppink"))

axis(1, at=seq(0,35000,5000), label=c("0", "5,000", "10,000", "15,000", "20,000", "25,000", "30,000", "35,000"), cex.axis=0.8)
axis(2, at=seq(0,5250,750), label=c("0", "750", "1500", "2250", "3,000", "3750", "4500", "5250"), las=2, cex.axis=0.8)
box()
mtext(side=2, "Number of OTUs", line=4)

dev.off()
```

\newpage
\doublespacing


## References
