```{r, echo=FALSE, eval=TRUE, warning=FALSE, message=FALSE}
opts_chunk$set("dev" = "png")
opts_chunk$set(results = "hold")
opts_chunk$set(fig.show = "hold")
opts_chunk$set(warning = FALSE)
opts_chunk$set(fig.align = "center")
opts_chunk$set(cache = FALSE)

opts_chunk$set(eval = FALSE)
opts_chunk$set(echo = FALSE)
```



## Get data out of ARB

> To create SSU Ref (ARB file), all sequences below 1,200 bases for Bacteria and
> Eukarya and below 900 bases for Archaea or an alignment identity below 70 or an
> alignment quality value below 50 have been removed from SSU Parc. All sequences
> with a Pintail value < 50 or an alignment quality value < 75 have been assigned
> to color group 1 in ARB (red). All Living Tree Project or  StrainInfo
> typestrains have been assigned to color group 2 in ARB (light blue). From
> http://www.arb-silva.de/documentation/release-119/


```{r get_data, engine="bash"}
wget http://www.arb-silva.de/fileadmin/silva_databases/release_119/ARB_files/SSURef_119_SILVA_14_07_14_opt.arb.tgz
wget http://www.arb-silva.de/fileadmin/silva_databases/release_119/ARB_files/SSURef_119_SILVA_14_07_14_opt.arb.tgz.md5
tar xvzf SSURef_119_SILVA_14_07_14_opt.arb.tgz
```

Within ARB, we will exclude color group 1, chloroplasts, and mitochondria.

To get good sequences...
* Go to search window
* Set Search_Fields to "ARB_color" and use "1"; click on the equal sign to make it not equal
* Hit Mark Listed, Unmark Rest button (N=1493493)
* The problem with the taxonomies is that the sequences don't all have taxonomies. Need to figure
    out which taxonomy to base the analysis on. Do the following searches...
    * ARB_color != 1 & tax_rdp == "*Bacteria*" (N=1324558)
    * ARB_color != 1 & tax_greengenes == "*Bacteria*" (N=1123144)
    * ARB_color != 1 & tax_slv == "*Bacteria*" (N=426917)
    * ARB_color != 1 & tax_embl == "*Bacteria*" (N=1207210)
    * tax_rdp | tax_greengenes | tax_embl | tax_slv (N=1376910)


Let's stick with the bacterial sequences that have an RDP taxonomy and we will
analyze them in RDP space, but preserve the taxonomy generated by the other
systems. Nicely, the RDP taxonomy does not include mitochondrial
16S rRNA gene sequences, so all we'll need to remove are the chloroplast
sequences

* Return to Search and Query window
* Change search field to "ARB_color", enter "1", and click the equal sign to be not-equal
* Change the second search field to "tax_rdp" and set it to "*Bacteria*"
* Change the third search field to "tax_rdp" and set it to "*Chloroplast*"
* Set the two Query operators to "and" and hit "Search"
* Hit Mark Listed Unmark Rest (N=1,301,541)
* Click "Write to Fields of Listed", select the "remark" field, and enter "good bacteria"
* Click "Write"
* In main ARB window go File->Export->Export to external format
* Select Compress -> "Vertical Gaps" and fasta_mothur.eft as the format
* Rename `noname.fasta` to `bacteria.fasta`

Now we need the taxonomy information.
* Go Tree -> NDS
* Click "name", "acc", "tax_rdp". The "tax_rdp" field should have 250 characters
* Unclick everything else
* Click "Close"
* Go File->Export->Export fields
* Set the file name to "bacteria.taxonomy" and Column output to "TAB separated"
* Click "SAVE"

Finally, let's save the database by doing File -> Quick Save Changes and then
quit out of ARB.


## Format taxonomy file

The next thing we need to do is to clean up the bacteria.taxonomy.nds file
to make it into a proper, mothur compatible bacteria.taxonomy file. First we
want to remove the names of any sub taxa (e.g. suborder). To do this we need
to get the RDP list of names and the taxonomic level they belong to...

```{r get_rdp_names, engine="bash"}
rm -rf RDPClassifier_16S_trainsetNo10_rawtrainingdata* Trainset9_032012.pds.zip

wget -N http://sourceforge.net/projects/rdp-classifier/files/RDP_Classifier_TrainingData/RDPClassifier_16S_trainsetNo10_rawtrainingdata.zip
unzip -o RDPClassifier_16S_trainsetNo10_rawtrainingdata.zip
mv RDPClassifier_16S_trainsetNo10_rawtrainingdata/trainset10_db_taxid.txt ./
rm -rf RDPClassifier_16S_trainsetNo10_rawtrainingdata*
```

Now we're read to do some R'ing to get the taxonomy file formatted properly...

```{r format_taxonomy}
tax_data <- read.table(file="bacteria.taxonomy.nds", sep="\t")

#get the names to match the fasta file
seq_names <- paste(tax_data$V2, tax_data$V1, sep=".")

taxonomy <- gsub(" ", "_", tax_data$V3)  #convert any spaces to underscores
taxonomy <- gsub("[^;]*_incertae_sedis$", "", taxonomy) #remove the unknowns
taxonomy <- gsub('\"', '', taxonomy) #remove quote marks

#now we need to pull out the taxon names for the sub taxa levels
levels <- read.table(file="trainset10_db_taxid.txt", sep="*", stringsAsFactors=F)
subs <- levels[grep("sub", levels$V5),] #get the sub taxa names
sub.names <- subs$V2

tax.split <- strsplit(taxonomy, split=";")  #split the taxonomy string by semicolon

#function to see which taxa names in a list are found in the sub taxa list
remove.subs <- function(tax.vector){
,	#return(tax.vector[which(!tax.vector %in% sub.names)])
}

#apply sub taxa finding function to all sequences
no.subs <- lapply(tax.split, remove.subs)

#merge data together by semicolon for each sequence
no.subs.str <- unlist(lapply(no.subs, paste, collapse=";"))

#Tack a semicolon on to the end of each sequence
no.subs.str <- paste0(no.subs.str, ";")

write.table(cbind(seq_names, no.subs.str), "bacteria.tax", row.names=F,
                                                col.names=F, quote=F, sep="\t")
```



## Get good sequences

Now we need to know the start/end position of the sequences so that we can
make sure the reads overlap the same alignment space.


```{r get_summary, engine="bash", results="hide", echo=TRUE}
mothur "#summary.seqs(fasta=bacteria.fasta, processors=12)"
```

```
                Start   End     NBases  Ambigs  Polymer NumSeqs
Minimum:        1       9693    900     0       4       1
2.5%-tile:      1004    12895   1279    0       5       32539
25%-tile:       1026    13069   1355    0       5       325386
Median:         1044    13267   1382    0       5       650771
75%-tile:       1051    13825   1447    0       6       976156
97.5%-tile:     1793    13991   1522    2       7       1269003
Maximum:        6448    15767   3281    51      38      1301541
Mean:   1096.18 13442   1399.39 0.176343        5.49738
# of Seqs:      1301541
```


Notes...
* Looks like an archaeal sequence slipped through
* Will want to get rid of sequences with large number of Ns in them - how large?


```{r find_screening_points}
data <- read.table(file="bacteria.summary", header=T, row.names=1)
quantile(data$start, probs=seq(0.9,1,0.01))
start <- 2000

quantile(data$end, probs=seq(0,0.1,0.01))
end <- 13000

quantile(data$nbases, probs=seq(0.0,0.1,0.01))
min_length <- 1200

quantile(data$ambigs, probs=seq(0.9,1.0,0.01))
max_ambig <- 2

trimmed <- data[data$start <= start & data$end >= end &
                    data$nbases > min_length & data$ambigs <= max_ambig,]
nrow(trimmed)
summary(trimmed$nbases)
```

Now let's run these parameters using mothur.

```{r screen_data, engine="bash"}
mothur "#set.current(processors=8);
        set.dir(input=data/raw, output=data/process)
        screen.seqs(fasta=bacteria.fasta, taxonomy=bacteria.tax, start=2000, end=13000, maxambig=2, minlength=1200);"
```



## Single cell genomics data

Using 16S rRNA gene sequences from the single cell genomics projects we'd like
to see whether that type of data has had a meaningful impact on the trajectory
of the microbial census. The LSU team scraped these 251 sequences from a
database and have provided them to me as a fasta file. Some of them are quite
short and none of them are aligned. Let's go ahead and align them to the SILVA
alignment and then screen them to keep sequences that overlap with the SILVA
sequences.

```{r single_cell, engine="bash"}
mothur "#align.seqs(fasta=single_cell.bacteria.fasta, reference=bacteria.good.fasta);
        screen.seqs(fasta=current, start=2000, end=13000, maxambig=2, minlength=1200);
        classify.seqs(fasta=current, reference=trainset10_082014.pds.fasta, taxonomy=trainset10_082014.pds.tax, cutoff=80, processors=8)"
```


## Pooling the data

Now we want to merge the SILVA and single cell genomics 16S rRNA gene sequence
collections. We'll also filter the sequences to overlap in the same alignment
space, unique them, and precluster them...

```{r pool_data, engine="bash"}
cat bacteria.good.fasta single_cell.bacteria.good.align > all_bacteria.align
cat bacteria.good.tax single_cell.bacteria.good.pds.wang.taxonomy > all_bacteria.taxonomy

mothur "#filter.seqs(fasta=all_bacteria.align, vertical=T, trump=.);
        unique.seqs();
        pre.cluster(fasta=current, name=current, diffs=10);"

#need to make the taxonomy file match the fasta and names files...
cut -f 1 *ster.names > precluster.accnos
mothur "#get.seqs(taxonomy=all_bacteria.taxonomy, accnos=precluster.accnos)"
```

Now we're ready to cluster the sequences. We'll do it by the classic approach
without any cutoffs and see what we get. Let's start by splitting things at the
phylum level and cluster from there...

```{r cluster_data, engine="bash"}
mothur "#cluster.split(fasta=all_bacteria.filter.unique.precluster.fasta, name=all_bacteria.filter.unique.precluster.names, taxonomy=all_bacteria.pick.taxonomy, taxlevel=2, classic=T, processors=8)"
```


## Get metadata

We'd like to use some metadata from the database to characterize the changes in
the representation of sequences over time, by environment, methods, etc. The
fields housed within SILVA are available at their [website](http://www.arb-silva.de/fileadmin/arb_web_db/release_115/Fields_description/SILVA_description_of_fields_16_06_2013.htm)
and there is an [FAQ](http://www.arb-silva.de/documentation/faqs/) on their site
as well. There were a number of fields that I didn't think were relevant and
instead focused on 30 fields that I thought could help the cause. I marked these
fields in the NDS feature and extracted them to `metadata.nds`. We need to
tweak this file slightly in R to get it into a format that we can use.


```{r}
metadata_fields <- c(
    "name", #internal ARB database ID, do not change!
    "acc", #accession number
    "bio_material", #identifier for the biological material from which the nucleic acid sequenced was obtained
    "clone", #clone from which the sequence was obtained
    "clone_lib", #clone library from which the sequence was obtained
    "collected_by", #name of the person who collected the specimen
    "collection_date", #date that the sample/specimen was collected
    "country", #geographical origin of sequenced sample
    "culture_collection", #institution code and identifier for the culture from which the nucleic acid sequenced was obtained, with optional collection code
    "date", #entry creation and update date separated by ;
    "description", #description
    "embl_class", #describes the data class in EMBL, e.g. CON: Constructed, WGS: Whole Genome Shotgun, see www.ebi.ac.uk/ena/about/embl_bank_format
    "env_sample", #identifies sequences derived by direct molecular isolation from a bulk environmental DNA sample (by PCR with or without subsequent cloning of the product, DGGE, or other anonymous methods) with no reliable identification of the source organism. Indicated by ‘yes’ in the ARB files
    "full_name", #organism species
    "host", #natural (as opposed to laboratory) host to the organism from which sequenced molecule was obtained.
    "identified_by", #name of the taxonomist who identified the specimen
    "insdc", #the International Nucleotide Sequence Database Collaboration (INSDC) Project Identifier that has been assigned to the entry
    "isolate", #individual isolate from which the sequence was obtained
    "isolation_source", #describes the physical, environmental and/or local geographical source of the biological sample from which the sequence was derived
    "journal", #reference location
    "lat_lon", #geographical coordinates of the location where the specimen was collected
    "publication_doi", #cross-reference DOI number
    "pubmed_id", #cross-reference Pubmed ID
    "strain", #strain from which the sequence was obtained. (t) or [T]: typestrains, [C]: cultivated, [G]: genomes
    "sub_species", #name of sub-species of organism from which sequence was obtained
    "submit_author", #submission authors from reference location
    "submit_date", #submission date from reference location
    "title", #reference title
    "depth_slv", #depth is the vertical distance below surface, e.g. for sediment or soil samples depth is measured from sediment or soil surface, respectively. Depth can be reported as an interval for subsurface samples.
    "habitat_slv" #habitat description according to EnvO-Lite
)


metadata <- read.table(file="data/raw/metadata.nds", sep="\t", stringsAsFactors=FALSE,
                            col.names=metadata_fields, fill=TRUE, na.strings="",
                            quote="")
rownames(metadata) <- paste(metadata$name, metadata$acc, sep=".")
metadata <- metadata[,-c(1,2)]
```

Now we want to read in `bacteria.bad.accnos` so that we can figure out which
sequences to cull from the metadata table.

```{r}
bad_accnos_data <- read.table(file="data/process/bacteria.bad.accnos", col.names=c("name.accnos", "reason"))
bad_accnos <- bad_accnos_data$name.accnos
metadata_good <- metadata[-c(bad_accnos),]
write.table(metadata_good, file="data/process/all_bacteria.good.metadata", quote=F, sep="\t")
```

We'd also like to get the metadata from the single cell data. We have an `xlsx` that the LSU group pulled together that we'll read in and concatenate to the end of `bacteria.good.metadata`.

```{r}
if(!"openxlsx" %in% rownames(installed.packages())){
    install.packages("openxlsx")
}
library("openxlsx")

spread_sheet <- read.xlsx("data/raw/BacterialSSUMetadata.xlsx", sheet=1, startRow=1, colNames=TRUE)
bad_accnos_data <- read.table(file="data/process/single_cell.bacteria.bad.accnos", col.names=c("name.accnos", "reason"))
bad_accnos <- bad_accnos_data$name.accnos
spread_sheet_good <- spread_sheet[-c(bad_accnos),]

write.table(spread_sheet_good, file="data/process/all_bacteria.good.metadata", quote=F, sep="\t", col.names=F, append=T)
```
